diff --git a/cad/lepton-eda/Makefile b/cad/lepton-eda/Makefile
index c4d8239e892d..af7890d25180 100644
--- a/cad/lepton-eda/Makefile
+++ b/cad/lepton-eda/Makefile
@@ -1,8 +1,7 @@
 PORTNAME=	lepton-eda
-PORTVERSION=	1.9.17
-PORTREVISION=	1
+PORTVERSION=	1.9.18
 CATEGORIES=	cad
-MASTER_SITES=	https://github.com/lepton-eda/lepton-eda/releases/download/${PORTVERSION}-20211219/
+MASTER_SITES=	https://github.com/lepton-eda/lepton-eda/releases/download/${PORTVERSION}-20220529/
 
 MAINTAINER=	graahnul.grom@gmail.com
 COMMENT=	Lepton Electronic Design Automation
@@ -41,6 +40,7 @@ SHEBANG_FILES=	utils/pcb_backannotate/lepton-pcb_backannotate \
 		utils/symfix/lepton-symfix
 
 GNU_CONFIGURE=	yes
+CONFIGURE_ARGS=	--enable-guild
 INSTALL_TARGET=	install-strip
 
 INFO=	lepton-scheme lepton-manual
diff --git a/cad/lepton-eda/distinfo b/cad/lepton-eda/distinfo
index 6233f085cc22..b05218134d02 100644
--- a/cad/lepton-eda/distinfo
+++ b/cad/lepton-eda/distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1640711244
-SHA256 (lepton-eda-1.9.17.tar.gz) = 94e9de29e254730ea3397ff78aff410d6389df18a9fef1a1cf11c73406edb12f
-SIZE (lepton-eda-1.9.17.tar.gz) = 9454425
+TIMESTAMP = 1656147585
+SHA256 (lepton-eda-1.9.18.tar.gz) = 5fdc8dba8b0d47525fde061641978e9ca771cc92e5776f529fd5d8b0f1966182
+SIZE (lepton-eda-1.9.18.tar.gz) = 9459680
diff --git a/cad/lepton-eda/pkg-plist b/cad/lepton-eda/pkg-plist
index aec945e034bc..45757636ea28 100644
--- a/cad/lepton-eda/pkg-plist
+++ b/cad/lepton-eda/pkg-plist
@@ -41,6 +41,7 @@ include/liblepton/export.h
 include/liblepton/fill.h
 include/liblepton/forward.h
 include/liblepton/funcs.h
+include/liblepton/glib_compat.h
 include/liblepton/globals.h
 include/liblepton/liblepton.h
 include/liblepton/libleptonguile.h
@@ -69,14 +70,14 @@ include/liblepton/transform.h
 include/liblepton/undo.h
 include/libleptonattrib/prototype.h
 lib/liblepton.so
-lib/liblepton.so.10
-lib/liblepton.so.10.0.0
+lib/liblepton.so.11
+lib/liblepton.so.11.0.0
 lib/libleptonattrib.so
-lib/libleptonattrib.so.4
-lib/libleptonattrib.so.4.0.0
+lib/libleptonattrib.so.5
+lib/libleptonattrib.so.5.0.0
 lib/libleptongui.so
-lib/libleptongui.so.4
-lib/libleptongui.so.4.0.0
+lib/libleptongui.so.5
+lib/libleptongui.so.5.0.0
 libdata/pkgconfig/liblepton.pc
 libdata/pkgconfig/libleptonattrib.pc
 libdata/pkgconfig/libleptongui.pc
@@ -102,8 +103,6 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/7400-1-hidden.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/7400-1.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Add-Component-widget.html
-%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Add-attributes.html
-%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Adding-attributes-to-your-symbol.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Allegro-backend.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Attribute-compatibility.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Attributes.html
@@ -179,6 +178,7 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/PADS-backend.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Path-data.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Printing-and-exporting.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Program-Index.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Quitting-the-program.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Related-tools.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Resources.html
@@ -914,6 +914,141 @@ share/info/text-layout.png
 %%DATADIR%%/bitmap/gschem-top2bottom.png
 %%DATADIR%%/bitmap/gschem-undo.xpm
 %%DATADIR%%/bitmap/gschem-warning.png
+%%DATADIR%%/ccache/auto-refdes.go
+%%DATADIR%%/ccache/auto-uref.go
+%%DATADIR%%/ccache/color-map.go
+%%DATADIR%%/ccache/conf/schematic/attribs.go
+%%DATADIR%%/ccache/conf/schematic/deprecated.go
+%%DATADIR%%/ccache/conf/schematic/keys.go
+%%DATADIR%%/ccache/conf/schematic/menu.go
+%%DATADIR%%/ccache/conf/schematic/stroke.go
+%%DATADIR%%/ccache/geda-deprecated-config.go
+%%DATADIR%%/ccache/geda.go
+%%DATADIR%%/ccache/geda/attrib.go
+%%DATADIR%%/ccache/geda/config.go
+%%DATADIR%%/ccache/geda/deprecated.go
+%%DATADIR%%/ccache/geda/library.go
+%%DATADIR%%/ccache/geda/log.go
+%%DATADIR%%/ccache/geda/object.go
+%%DATADIR%%/ccache/geda/os.go
+%%DATADIR%%/ccache/geda/page.go
+%%DATADIR%%/ccache/gschem/action.go
+%%DATADIR%%/ccache/gschem/attrib.go
+%%DATADIR%%/ccache/gschem/builtins.go
+%%DATADIR%%/ccache/gschem/deprecated.go
+%%DATADIR%%/ccache/gschem/gschemdoc.go
+%%DATADIR%%/ccache/gschem/hook.go
+%%DATADIR%%/ccache/gschem/keymap.go
+%%DATADIR%%/ccache/gschem/selection.go
+%%DATADIR%%/ccache/gschem/symbol/check.go
+%%DATADIR%%/ccache/gschem/util.go
+%%DATADIR%%/ccache/gschem/window.go
+%%DATADIR%%/ccache/lepton/attrib.go
+%%DATADIR%%/ccache/lepton/autoplace.go
+%%DATADIR%%/ccache/lepton/color-map.go
+%%DATADIR%%/ccache/lepton/config.go
+%%DATADIR%%/ccache/lepton/eval.go
+%%DATADIR%%/ccache/lepton/ffi.go
+%%DATADIR%%/ccache/lepton/ffi/lib.go
+%%DATADIR%%/ccache/lepton/file-system.go
+%%DATADIR%%/ccache/lepton/gerror.go
+%%DATADIR%%/ccache/lepton/gettext.go
+%%DATADIR%%/ccache/lepton/legacy-config.go
+%%DATADIR%%/ccache/lepton/legacy-config/keylist.go
+%%DATADIR%%/ccache/lepton/library.go
+%%DATADIR%%/ccache/lepton/library/component.go
+%%DATADIR%%/ccache/lepton/log-rotate.go
+%%DATADIR%%/ccache/lepton/log.go
+%%DATADIR%%/ccache/lepton/m4.go
+%%DATADIR%%/ccache/lepton/object.go
+%%DATADIR%%/ccache/lepton/object/foreign.go
+%%DATADIR%%/ccache/lepton/object/text.go
+%%DATADIR%%/ccache/lepton/object/type.go
+%%DATADIR%%/ccache/lepton/option.go
+%%DATADIR%%/ccache/lepton/os.go
+%%DATADIR%%/ccache/lepton/page.go
+%%DATADIR%%/ccache/lepton/page/foreign.go
+%%DATADIR%%/ccache/lepton/rc.go
+%%DATADIR%%/ccache/lepton/repl.go
+%%DATADIR%%/ccache/lepton/srfi-37.go
+%%DATADIR%%/ccache/lepton/toplevel.go
+%%DATADIR%%/ccache/lepton/version.go
+%%DATADIR%%/ccache/netlist.go
+%%DATADIR%%/ccache/netlist/attrib/compare.go
+%%DATADIR%%/ccache/netlist/attrib/refdes.go
+%%DATADIR%%/ccache/netlist/backend-getopt.go
+%%DATADIR%%/ccache/netlist/config.go
+%%DATADIR%%/ccache/netlist/deprecated.go
+%%DATADIR%%/ccache/netlist/duplicate.go
+%%DATADIR%%/ccache/netlist/error.go
+%%DATADIR%%/ccache/netlist/hierarchy.go
+%%DATADIR%%/ccache/netlist/mode.go
+%%DATADIR%%/ccache/netlist/net.go
+%%DATADIR%%/ccache/netlist/option.go
+%%DATADIR%%/ccache/netlist/package-pin.go
+%%DATADIR%%/ccache/netlist/package.go
+%%DATADIR%%/ccache/netlist/partlist.go
+%%DATADIR%%/ccache/netlist/partlist/common.go
+%%DATADIR%%/ccache/netlist/port.go
+%%DATADIR%%/ccache/netlist/schematic-component.go
+%%DATADIR%%/ccache/netlist/schematic-connection.go
+%%DATADIR%%/ccache/netlist/schematic-port.go
+%%DATADIR%%/ccache/netlist/schematic.go
+%%DATADIR%%/ccache/netlist/schematic/toplevel.go
+%%DATADIR%%/ccache/netlist/subschematic-connection.go
+%%DATADIR%%/ccache/netlist/subschematic.go
+%%DATADIR%%/ccache/netlist/verbose.go
+%%DATADIR%%/ccache/pcb.go
+%%DATADIR%%/ccache/schematic/action.go
+%%DATADIR%%/ccache/schematic/attrib.go
+%%DATADIR%%/ccache/schematic/builtins.go
+%%DATADIR%%/ccache/schematic/core/gettext.go
+%%DATADIR%%/ccache/schematic/dialog.go
+%%DATADIR%%/ccache/schematic/doc.go
+%%DATADIR%%/ccache/schematic/ffi.go
+%%DATADIR%%/ccache/schematic/ffi/gobject.go
+%%DATADIR%%/ccache/schematic/ffi/gtk.go
+%%DATADIR%%/ccache/schematic/gui/keymap.go
+%%DATADIR%%/ccache/schematic/gui/stroke.go
+%%DATADIR%%/ccache/schematic/hook.go
+%%DATADIR%%/ccache/schematic/keymap.go
+%%DATADIR%%/ccache/schematic/menu.go
+%%DATADIR%%/ccache/schematic/netlist.go
+%%DATADIR%%/ccache/schematic/precompile.go
+%%DATADIR%%/ccache/schematic/repl.go
+%%DATADIR%%/ccache/schematic/selection.go
+%%DATADIR%%/ccache/schematic/symbol/check.go
+%%DATADIR%%/ccache/schematic/undo.go
+%%DATADIR%%/ccache/schematic/util.go
+%%DATADIR%%/ccache/schematic/window.go
+%%DATADIR%%/ccache/symbol/blame.go
+%%DATADIR%%/ccache/symbol/check.go
+%%DATADIR%%/ccache/symbol/check/alignment.go
+%%DATADIR%%/ccache/symbol/check/arc.go
+%%DATADIR%%/ccache/symbol/check/attrib.go
+%%DATADIR%%/ccache/symbol/check/box.go
+%%DATADIR%%/ccache/symbol/check/bus.go
+%%DATADIR%%/ccache/symbol/check/circle.go
+%%DATADIR%%/ccache/symbol/check/component.go
+%%DATADIR%%/ccache/symbol/check/connection.go
+%%DATADIR%%/ccache/symbol/check/duplicate.go
+%%DATADIR%%/ccache/symbol/check/entity-pin.go
+%%DATADIR%%/ccache/symbol/check/forbidden.go
+%%DATADIR%%/ccache/symbol/check/line.go
+%%DATADIR%%/ccache/symbol/check/log.go
+%%DATADIR%%/ccache/symbol/check/net-attrib.go
+%%DATADIR%%/ccache/symbol/check/net.go
+%%DATADIR%%/ccache/symbol/check/obsolete.go
+%%DATADIR%%/ccache/symbol/check/path.go
+%%DATADIR%%/ccache/symbol/check/picture.go
+%%DATADIR%%/ccache/symbol/check/pin-attrib.go
+%%DATADIR%%/ccache/symbol/check/pin.go
+%%DATADIR%%/ccache/symbol/check/primitive.go
+%%DATADIR%%/ccache/symbol/check/slot.go
+%%DATADIR%%/ccache/symbol/check/text.go
+%%DATADIR%%/ccache/symcheck/check.go
+%%DATADIR%%/ccache/symcheck/option.go
+%%DATADIR%%/ccache/symcheck/report.go
 %%DATADIR%%/geda-system.conf
 %%DATADIR%%/gschem-colormap-bw
 %%DATADIR%%/gschem-colormap-darkbg
@@ -948,8 +1083,6 @@ share/info/text-layout.png
 %%DATADIR%%/lepton-system.conf
 %%DATADIR%%/print-colormap-darkbg
 %%DATADIR%%/print-colormap-lightbg
-%%DATADIR%%/scheme/auto-place-attribs.scm
-%%DATADIR%%/scheme/auto-place-netname.scm
 %%DATADIR%%/scheme/auto-refdes.scm
 %%DATADIR%%/scheme/auto-uref.scm
 %%DATADIR%%/scheme/autoload/config-netlist.scm
@@ -999,7 +1132,6 @@ share/info/text-layout.png
 %%DATADIR%%/scheme/conf/schematic/keys.scm
 %%DATADIR%%/scheme/conf/schematic/menu.scm
 %%DATADIR%%/scheme/conf/schematic/stroke.scm
-%%DATADIR%%/scheme/default-attrib-positions.scm
 %%DATADIR%%/scheme/geda-deprecated-config.scm
 %%DATADIR%%/scheme/geda.scm
 %%DATADIR%%/scheme/geda/attrib.scm
@@ -1022,13 +1154,15 @@ share/info/text-layout.png
 %%DATADIR%%/scheme/gschem/util.scm
 %%DATADIR%%/scheme/gschem/window.scm
 %%DATADIR%%/scheme/lepton/attrib.scm
+%%DATADIR%%/scheme/lepton/autoplace.scm
 %%DATADIR%%/scheme/lepton/color-map.scm
 %%DATADIR%%/scheme/lepton/config.scm
-%%DATADIR%%/scheme/lepton/core/gettext.scm
 %%DATADIR%%/scheme/lepton/eval.scm
 %%DATADIR%%/scheme/lepton/ffi.scm
 %%DATADIR%%/scheme/lepton/ffi/lib.scm
 %%DATADIR%%/scheme/lepton/file-system.scm
+%%DATADIR%%/scheme/lepton/gerror.scm
+%%DATADIR%%/scheme/lepton/gettext.scm
 %%DATADIR%%/scheme/lepton/legacy-config.scm
 %%DATADIR%%/scheme/lepton/legacy-config/keylist.scm
 %%DATADIR%%/scheme/lepton/library.scm
@@ -1038,6 +1172,7 @@ share/info/text-layout.png
 %%DATADIR%%/scheme/lepton/m4.scm
 %%DATADIR%%/scheme/lepton/object.scm
 %%DATADIR%%/scheme/lepton/object/foreign.scm
+%%DATADIR%%/scheme/lepton/object/text.scm
 %%DATADIR%%/scheme/lepton/object/type.scm
 %%DATADIR%%/scheme/lepton/option.scm
 %%DATADIR%%/scheme/lepton/os.scm
@@ -1046,6 +1181,7 @@ share/info/text-layout.png
 %%DATADIR%%/scheme/lepton/rc.scm
 %%DATADIR%%/scheme/lepton/repl.scm
 %%DATADIR%%/scheme/lepton/srfi-37.scm
+%%DATADIR%%/scheme/lepton/toplevel.scm
 %%DATADIR%%/scheme/lepton/version.scm
 %%DATADIR%%/scheme/netlist.scm
 %%DATADIR%%/scheme/netlist/attrib/compare.scm
@@ -1078,10 +1214,10 @@ share/info/text-layout.png
 %%DATADIR%%/scheme/schematic/builtins.scm
 %%DATADIR%%/scheme/schematic/core/gettext.scm
 %%DATADIR%%/scheme/schematic/dialog.scm
+%%DATADIR%%/scheme/schematic/doc.scm
 %%DATADIR%%/scheme/schematic/ffi.scm
 %%DATADIR%%/scheme/schematic/ffi/gobject.scm
 %%DATADIR%%/scheme/schematic/ffi/gtk.scm
-%%DATADIR%%/scheme/schematic/gschemdoc.scm
 %%DATADIR%%/scheme/schematic/gui/keymap.scm
 %%DATADIR%%/scheme/schematic/gui/stroke.scm
 %%DATADIR%%/scheme/schematic/hook.scm
