diff --git a/cad/lepton-eda/Makefile b/cad/lepton-eda/Makefile
index 11bf7e7bc2f2..32898ad8cc5e 100644
--- a/cad/lepton-eda/Makefile
+++ b/cad/lepton-eda/Makefile
@@ -1,7 +1,7 @@
 PORTNAME=	lepton-eda
-PORTVERSION=	1.9.15
+PORTVERSION=	1.9.16
 CATEGORIES=	cad
-MASTER_SITES=	https://github.com/lepton-eda/lepton-eda/releases/download/${PORTVERSION}-20210626/
+MASTER_SITES=	https://github.com/lepton-eda/lepton-eda/releases/download/${PORTVERSION}-20210731/
 
 MAINTAINER=	graahnul.grom@gmail.com
 COMMENT=	Lepton Electronic Design Automation
diff --git a/cad/lepton-eda/distinfo b/cad/lepton-eda/distinfo
index c18f34a4591e..0476c44befe7 100644
--- a/cad/lepton-eda/distinfo
+++ b/cad/lepton-eda/distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1624755806
-SHA256 (lepton-eda-1.9.15.tar.gz) = bf7e0ced215054891c0ffcc7639530fef4e7122e02830af9dd80fbede2ab6845
-SIZE (lepton-eda-1.9.15.tar.gz) = 9471503
+TIMESTAMP = 1631290180
+SHA256 (lepton-eda-1.9.16.tar.gz) = c5d275d4ce115cd17a78f643e98fbf0d474edb50052dbc9d65cabd9d6834f989
+SIZE (lepton-eda-1.9.16.tar.gz) = 9493429
diff --git a/cad/lepton-eda/pkg-plist b/cad/lepton-eda/pkg-plist
index 8f1e998e4093..935c3fc22b76 100644
--- a/cad/lepton-eda/pkg-plist
+++ b/cad/lepton-eda/pkg-plist
@@ -73,7 +73,7 @@ lib/libleptonattrib.so.3
 lib/libleptonattrib.so.3.0.0
 lib/libleptongui.so
 lib/libleptongui.so.3
-lib/libleptongui.so.3.0.0
+lib/libleptongui.so.3.0.1
 libdata/pkgconfig/liblepton.pc
 libdata/pkgconfig/libleptonattrib.pc
 libdata/pkgconfig/libleptongui.pc
@@ -95,11 +95,17 @@ share/applications/lepton-attrib.desktop
 share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/NEWS.md
 %%PORTDOCS%%%%DOCSDIR%%/README.md
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/7400-1-hidden.png
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/7400-1.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Add-Component-widget.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Add-attributes.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Adding-attributes-to-your-symbol.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Allegro-backend.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Attributes.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/BOM-backend.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Back-annotation-from-PADS-PowerPCB.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Box-selection.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Cascade-backend.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Colors.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Communication.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Compatibility-with-geda_002dgaf.html
@@ -122,9 +128,12 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Dialog-boxes-and-widgets.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Documentation.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Editing-schematics.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Editing-symbols.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Editing-symversion-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Embedded-Components.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/File-format-overview.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Filenames.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Forward-annotation-to-PADS-PowerPCB.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Installation-from-git.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Installation-from-source.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Installation-of-lepton_002dattrib.html
@@ -138,6 +147,8 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Log-window.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Logging.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Macro-widget.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Master-Attribute-Document-Overview.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Master-Attribute-Document-Revision-History.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Menu-Add.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Menu-Attributes.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Menu-Edit.html
@@ -153,8 +164,10 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/New-configuration-system.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/No_002dconnect-symbols.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Object-types.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Obsolete-Attributes.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Other-resources.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Overview.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/PADS-backend.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Path-data.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Printing-and-exporting.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Quitting-the-program.html
@@ -163,9 +176,16 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Running-lepton_002dschematic.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Save-As-dialog-box.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Saving-schematic-pages.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Schematic-only-Attributes.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Selecting-and-deselecting-all-objects.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Selecting-objects.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Simple-object-selection.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Supported-operating-systems.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Switcap-backend.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Symbol-Creation.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Symbol-and-Schematic-Attributes.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Symbol-example.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Symbol-only-Attributes.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Symbols.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Title-block-symbol.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Topic-Index.html
@@ -173,12 +193,18 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Usage-of-lepton_002dattrib.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Utilities.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Web-pages.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/What-are-Attributes_003f.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Working-with-windows.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/analysis.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/arc.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/author-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/box.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/bus.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/cache_002dconfiguration_002dcontext.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/circle.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/ckt.png
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/clocks.png
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/comment-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component_002dlibrary.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component_002dlibrary_002dcommand.html
@@ -187,13 +213,30 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/coordinate-space.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/default_002dfilename.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/default_002dtitleblock.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/description-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/device-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/dist_002dlicense-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/documentation-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/email-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/export-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003acascade1.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003acascade2.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003aswitcap1.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003aswitcap2.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003aswitcap3.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003aswitcap4.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003aswitcap5.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003aswitcap6.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/font.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/footprint-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gEDA-File-Format-Document-Revision-History.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gEDA-file-format.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gEDA_002fgaf-Master-Attribute-Document.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gafrc.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/graphical-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gschemrc.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/index.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/label-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton-manual.css
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dattrib-menus.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dattrib.html
@@ -210,15 +253,30 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dschematic.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/line.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/max_002drecent_002dfiles.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/name-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net-attribute.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net_003d-attribute-mini_002dHOWTO.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/netlist-group.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/netlist_002ehierarchy-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/netname-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/numslots-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/path-example.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/path.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/picture.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/pin.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/pin_0023-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/pinlabel-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/pinnumber-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/pinseq-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/pintype-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/project_002dconfiguration.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/refdes-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/refdes-naming-conventions.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/refdes-naming-restrictions.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/reset_002dcomponent_002dlibrary.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/rf_cascade.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic-group.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002eattrib-group.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002ebackup-group.html
@@ -230,12 +288,22 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002estatus_002dbar-group.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002etabs-group.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002eundo-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/slot-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/slot_0023-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/slotdef-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/source-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/symversion-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/table_003afonts.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/table_003apintype.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/text-and-attributes.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/text-layout.png
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/type-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/undo_002dlevels.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/uref-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/use_002ddocks.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/use_002dlicense-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/user_002dconfiguration.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/value-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/version.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Actions.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Arcs.html
@@ -321,10 +389,8 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-covered_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-csygas.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-csygas.ru.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-cygwin.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-data_plotting_improvements.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-data_structure_design_discussion.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-debian.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-design_flow_and_hierarchy_roadmap.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-devel-tips.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-developer.html
@@ -342,12 +408,6 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-faq-pcb.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-faq-simulation.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-faq.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-fbabgapp.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-fc1.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-fc2.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-fc3.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-fc4.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-fedora_rpm_installation.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-format_translation.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gaf.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gaf_building_git_version.html
@@ -377,13 +437,11 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gparts_ii.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gparts_tc.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gparts_ug.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-grcsan.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-grenum_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gsch2pcb_readme.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gsch2pcb_tutorial.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gsch2pcb_tutorial_releasenotes.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gsch2pcb_tutorial_releasenotes.ru.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gschem_symbol_creation.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gschem_symbol_generators.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gschem_ug-attributes.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gschem_ug-autonumbering.html
@@ -436,9 +494,7 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-license.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-links.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-mailinglists.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-master_attributes_list.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-mcalc_readme.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-na_howto.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-ngnutmeg_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-ngsconvert_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-ngspice_and_gschem.html
@@ -461,13 +517,8 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-sn_readme.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-spice_improvements.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-spice_polys.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-ssan.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-start.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-style_guide.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-suse_10.0.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-suse_10.1.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-suse_9.3.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-suse_rpm_installation.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-syntax_features.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-systemc_netlister_readme.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-tasks.html
@@ -476,7 +527,6 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-tragesym_readme.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-tragesym_tutorial.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-transistor_guide.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-ubuntu.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-usage.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-vams_netlister_readme.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-verilog_netlister_readme.html
@@ -582,15 +632,12 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/gschem-sym-2.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/gschem-sym-3.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/gschem_screenshot_001.jpg
-%%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/net.jpg
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/one-sch-1.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/one-sch-2.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/one-sch-3.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/path_example_and_gate-smaller.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/pcb-el-1.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/pcb-el-2.png
-%%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/rf_cascade_figure1.jpg
-%%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/rf_cascade_figure2.jpg
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/schematicfinished2.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/screenshot_gerbv.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/screenshot_gerbv_thumb.png
@@ -606,12 +653,6 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/summing.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/summing.sch
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/suse_electronics_menu.png
-%%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/switcap_figure1.jpg
-%%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/switcap_figure2.jpg
-%%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/switcap_figure3.jpg
-%%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/switcap_figure4.jpg
-%%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/symbol_7400-1.png
-%%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/symbol_7400-1_hidden.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/two-sch-1.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/two-sch-2.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/warmup_add_component.png
@@ -776,8 +817,15 @@ share/icons/hicolor/scalable/apps/lepton-schematic.svg
 share/icons/hicolor/scalable/mimetypes/application-x-lepton-sch2pcb-project.svg
 share/icons/hicolor/scalable/mimetypes/application-x-lepton-schematic.svg
 share/icons/hicolor/scalable/mimetypes/application-x-lepton-symbol.svg
+share/info/7400-1-hidden.png
+share/info/7400-1.png
+share/info/analysis.png
+share/info/ckt.png
+share/info/clocks.png
 share/info/coordinate-space.png
+share/info/net-attribute.png
 share/info/path-example.png
+share/info/rf_cascade.png
 share/info/text-layout.png
 %%DATADIR%%/bitmap/gschem-about-logo.png
 %%DATADIR%%/bitmap/gschem-alignment_bottomcenter.png
