Index: cad/lepton-eda/Makefile
===================================================================
--- cad/lepton-eda/Makefile	(revision 526543)
+++ cad/lepton-eda/Makefile	(working copy)
@@ -1,8 +1,8 @@
 # $FreeBSD$
 
 PORTNAME=	lepton-eda
-PORTVERSION=	1.9.9
-DISTVERSIONSUFFIX=	-20191003
+PORTVERSION=	1.9.10
+DISTVERSIONSUFFIX=	-20200319
 CATEGORIES=	cad
 
 MAINTAINER=	graahnul.grom@gmail.com
@@ -19,19 +19,14 @@
 
 USES=		autoreconf desktop-file-utils gettext-tools gnome \
 		groff:build libtool localbase makeinfo pathfix perl5 \
-		pkgconfig python:2.7,run shared-mime-info shebangfix
+		pkgconfig shared-mime-info shebangfix
 USE_GITHUB=	yes
 USE_GNOME=	cairo gtk20
 USE_LDCONFIG=	yes
 
-SHEBANG_FILES=	utils/scripts/lepton-tragesym \
-		utils/scripts/lepton-archive.py \
-		utils/scripts/gschupdate \
-		utils/scripts/gsymupdate \
-		utils/scripts/lepton-pcb_backannotate \
+SHEBANG_FILES=	utils/scripts/lepton-pcb_backannotate \
 		utils/scripts/lepton-refdes_renum \
-		utils/scripts/lepton-symfix \
-		examples/lightning_detector/bom
+		utils/scripts/lepton-symfix
 
 PORTEXAMPLES=	examples/*
 
@@ -48,4 +43,13 @@
 NLS_CONFIGURE_ENABLE=	nls
 NLS_USES=		gettext-runtime
 
+post-patch:
+	${REINPLACE_CMD} -e \
+		's,\$$(docdir)/examples,\$$(datadir)/examples/lepton-eda,' \
+		${WRKSRC}/examples/Makefile.am \
+		${WRKSRC}/examples/RF_Amp/Makefile.am \
+		${WRKSRC}/examples/TwoStageAmp/Makefile.am \
+		${WRKSRC}/examples/gTAG/Makefile.am \
+		${WRKSRC}/examples/lightning_detector/Makefile.am
+
 .include <bsd.port.mk>
Index: cad/lepton-eda/distinfo
===================================================================
--- cad/lepton-eda/distinfo	(revision 526543)
+++ cad/lepton-eda/distinfo	(working copy)
@@ -1,3 +1,3 @@
-TIMESTAMP = 1570203233
-SHA256 (lepton-eda-lepton-eda-1.9.9-20191003_GH0.tar.gz) = 13448d91ae17ab8c5e5cbba56c5a4e6a7cc3b3cb392280cb4d218f50891469d7
-SIZE (lepton-eda-lepton-eda-1.9.9-20191003_GH0.tar.gz) = 8361951
+TIMESTAMP = 1585112125
+SHA256 (lepton-eda-lepton-eda-1.9.10-20200319_GH0.tar.gz) = 271fac1424a963117dfe687846c8d884bc70ed8bf3a50c72ad5cc00741a5c403
+SIZE (lepton-eda-lepton-eda-1.9.10-20200319_GH0.tar.gz) = 8386076
Index: cad/lepton-eda/files/patch-build-tools_icon-theme-installer
===================================================================
--- cad/lepton-eda/files/patch-build-tools_icon-theme-installer	(revision 526543)
+++ cad/lepton-eda/files/patch-build-tools_icon-theme-installer	(nonexistent)
@@ -1,20 +0,0 @@
---- build-tools/icon-theme-installer.orig	2019-10-03 20:45:55 UTC
-+++ build-tools/icon-theme-installer
-@@ -103,13 +103,13 @@ if test -z "$INSTALL_BASE_DIR"; then
- 	exit 1
- fi
- 
--if test ! -x `echo "$MKINSTALLDIRS_EXEC" | cut -f1 -d' '`; then
--	echo "Cannot find '$MKINSTALLDIRS_EXEC'; You probably want to pass -m \$(mkinstalldirs)"
-+if test -z "$MKINSTALLDIRS_EXEC"; then
-+	echo "\$MKINSTALLDIRS_EXEC is not set. Pass it with the -m command line option"
- 	exit 1
- fi
- 
--if test ! -x `echo "$INSTALL_DATA_EXEC" | cut -f1 -d' '`; then
--	echo "Cannot find '$INSTALL_DATA_EXEC'; You probably want to pass -x \$(INSTALL_DATA)"
-+if test -z "$INSTALL_DATA_EXEC"; then
-+	echo "\$INSTALL_DATA_EXEC' is not set. Pass it with the -x command line option"
- 	exit 1
- fi
- 

Property changes on: cad/lepton-eda/files/patch-build-tools_icon-theme-installer
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-docs_scheme-api_Makefile.am
===================================================================
--- cad/lepton-eda/files/patch-docs_scheme-api_Makefile.am	(revision 526543)
+++ cad/lepton-eda/files/patch-docs_scheme-api_Makefile.am	(nonexistent)
@@ -1,14 +0,0 @@
---- docs/scheme-api/Makefile.am.orig	2019-10-09 16:39:51 UTC
-+++ docs/scheme-api/Makefile.am
-@@ -4,10 +4,8 @@ AM_MAKEINFOHTMLFLAGS = --css-ref=lepton-
- 
- EXTRA_DIST = lepton-scheme.css
- 
--html-local:
-+all: all-am html
- 	$(MKDIR_P) $(builddir)/lepton-scheme.html/
- 	cp -fv $(srcdir)/lepton-scheme.css $(builddir)/lepton-scheme.html/
- 
--all-local: html
--
- install-data-local: install-html

Property changes on: cad/lepton-eda/files/patch-docs_scheme-api_Makefile.am
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-examples_Makefile.am
===================================================================
--- cad/lepton-eda/files/patch-examples_Makefile.am	(revision 526543)
+++ cad/lepton-eda/files/patch-examples_Makefile.am	(nonexistent)
@@ -1,24 +0,0 @@
---- examples/Makefile.am.orig	2019-02-23 08:55:55 UTC
-+++ examples/Makefile.am
-@@ -1,9 +1,9 @@
- 
- SUBDIRS = gTAG lightning_detector RF_Amp TwoStageAmp
- 
--exampledir = $(docdir)/examples/
-+exampledir = $(datadir)/examples/lepton-eda/
- 
--example_DATA = README 
-+example_DATA = README
- 
- EXTRA_DIST = ChangeLog ChangeLog-1.0 $(example_DATA)
- 
-@@ -16,7 +16,7 @@ ChangeLog: $(top_builddir)/stamp-git
- 	) > $@
- endif HAVE_GIT_REPO
- 
--MOSTLYCLEANFILES = *.log core FILE *~ 
--CLEANFILES = *.log core FILE *~ 
-+MOSTLYCLEANFILES = *.log core FILE *~
-+CLEANFILES = *.log core FILE *~
- DISTCLEANFILES = *.log core FILE *~
- MAINTAINERCLEANFILES = *.log core FILE *~ Makefile.in ChangeLog

Property changes on: cad/lepton-eda/files/patch-examples_Makefile.am
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-examples_RF__Amp_Makefile.am
===================================================================
--- cad/lepton-eda/files/patch-examples_RF__Amp_Makefile.am	(revision 526543)
+++ cad/lepton-eda/files/patch-examples_RF__Amp_Makefile.am	(nonexistent)
@@ -1,11 +0,0 @@
---- examples/RF_Amp/Makefile.am.orig	2019-02-23 08:55:55 UTC
-+++ examples/RF_Amp/Makefile.am
-@@ -1,7 +1,7 @@
- 
- SUBDIRS = model sym
- 
--exampledir = $(docdir)/examples/RF_Amp/
-+exampledir = $(datadir)/examples/lepton-eda/RF_Amp/
- 
- example_DATA = 5980-2396E.pdf MSA-2643.cir MSA-2643.sch \
- 	Q1.cir Q1.sch Q2.cir Q2.sch README Simulation.cmd gafrc

Property changes on: cad/lepton-eda/files/patch-examples_RF__Amp_Makefile.am
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-examples_RF__Amp_model_Makefile.am
===================================================================
--- cad/lepton-eda/files/patch-examples_RF__Amp_model_Makefile.am	(revision 526543)
+++ cad/lepton-eda/files/patch-examples_RF__Amp_model_Makefile.am	(nonexistent)
@@ -1,12 +0,0 @@
---- examples/RF_Amp/model/Makefile.am.orig	2019-02-23 08:55:55 UTC
-+++ examples/RF_Amp/model/Makefile.am
-@@ -1,7 +1,7 @@
--exampledir = $(docdir)/examples/RF_Amp/model/
-+exampledir = $(datadir)/examples/lepton-eda/RF_Amp/model/
- 
- example_DATA = BJTM1_Q1.mod BJTM1_Q2.mod DiodeM1_Q1.mod DiodeM1_Q2.mod \
--             DiodeM2_Q1.mod DiodeM2_Q2.mod DiodeM3_Q1.mod DiodeM3_Q2.mod 
-+             DiodeM2_Q1.mod DiodeM2_Q2.mod DiodeM3_Q1.mod DiodeM3_Q2.mod
- 
- EXTRA_DIST = $(example_DATA)
- 

Property changes on: cad/lepton-eda/files/patch-examples_RF__Amp_model_Makefile.am
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-examples_RF__Amp_sym_Makefile.am
===================================================================
--- cad/lepton-eda/files/patch-examples_RF__Amp_sym_Makefile.am	(revision 526543)
+++ cad/lepton-eda/files/patch-examples_RF__Amp_sym_Makefile.am	(nonexistent)
@@ -1,8 +0,0 @@
---- examples/RF_Amp/sym/Makefile.am.orig	2019-02-23 08:55:55 UTC
-+++ examples/RF_Amp/sym/Makefile.am
-@@ -1,4 +1,4 @@
--exampledir = $(docdir)/examples/RF_Amp/sym/
-+exampledir = $(datadir)/examples/lepton-eda/RF_Amp/sym/
- 
- example_DATA = BJT_Model.sym Q_Model.sym spice-subcircuit-IO-1.sym \
- 	     spice-subcircuit-LL-1.sym

Property changes on: cad/lepton-eda/files/patch-examples_RF__Amp_sym_Makefile.am
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-examples_TwoStageAmp_Makefile.am
===================================================================
--- cad/lepton-eda/files/patch-examples_TwoStageAmp_Makefile.am	(revision 526543)
+++ cad/lepton-eda/files/patch-examples_TwoStageAmp_Makefile.am	(nonexistent)
@@ -1,8 +0,0 @@
---- examples/TwoStageAmp/Makefile.am.orig	2019-02-23 08:55:55 UTC
-+++ examples/TwoStageAmp/Makefile.am
-@@ -1,4 +1,4 @@
--exampledir = $(docdir)/examples/TwoStageAmp/
-+exampledir = $(datadir)/examples/lepton-eda/TwoStageAmp/
- 
- nobase_example_DATA = \
- 	Simulation.cmd TwoStageAmp.sch spice.netlist gafrc \

Property changes on: cad/lepton-eda/files/patch-examples_TwoStageAmp_Makefile.am
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-examples_gTAG_Makefile.am
===================================================================
--- cad/lepton-eda/files/patch-examples_gTAG_Makefile.am	(revision 526543)
+++ cad/lepton-eda/files/patch-examples_gTAG_Makefile.am	(nonexistent)
@@ -1,8 +0,0 @@
---- examples/gTAG/Makefile.am.orig	2019-02-23 08:55:55 UTC
-+++ examples/gTAG/Makefile.am
-@@ -1,4 +1,4 @@
--exampledir = $(docdir)/examples/gTAG/
-+exampledir = $(datadir)/examples/lepton-eda/gTAG/
- 
- example_DATA = 7414-1.sym attribs gafrc copyleft.sym crdist.sh \
- 	     cy7c64603-52nc.sym gTAG-consio.ps gTAG-consio.sch \

Property changes on: cad/lepton-eda/files/patch-examples_gTAG_Makefile.am
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-examples_lightning__detector_Makefile.am
===================================================================
--- cad/lepton-eda/files/patch-examples_lightning__detector_Makefile.am	(revision 526543)
+++ cad/lepton-eda/files/patch-examples_lightning__detector_Makefile.am	(nonexistent)
@@ -1,16 +0,0 @@
---- examples/lightning_detector/Makefile.am.orig	2019-02-23 08:55:55 UTC
-+++ examples/lightning_detector/Makefile.am
-@@ -1,11 +1,11 @@
--exampledir = $(docdir)/examples/lightning_detector/
-+exampledir = $(datadir)/examples/lepton-eda/lightning_detector/
- 
- nobase_example_DATA = \
- 	bom bom.readme lightning.net lightning.pcb lightning.sch \
- 	lightning.sch.bom gafrc \
- 	sym/2N4401.sym sym/2N4403.sym
- 
--EXTRA_DIST = $(nobase_example_DATA) 
-+EXTRA_DIST = $(nobase_example_DATA)
- 
- MOSTLYCLEANFILES = gschem.log *~
- CLEANFILES = gschem.log *~

Property changes on: cad/lepton-eda/files/patch-examples_lightning__detector_Makefile.am
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-liblepton_include_liblepton_geda__list.h
===================================================================
--- cad/lepton-eda/files/patch-liblepton_include_liblepton_geda__list.h	(revision 526543)
+++ cad/lepton-eda/files/patch-liblepton_include_liblepton_geda__list.h	(nonexistent)
@@ -1,23 +0,0 @@
---- liblepton/include/liblepton/geda_list.h.orig	2019-10-03 20:45:55 UTC
-+++ liblepton/include/liblepton/geda_list.h
-@@ -1,7 +1,8 @@
--/* gEDA - GPL Electronic Design Automation
-- * libgeda - gEDA's library
-+/* Lepton EDA library
-  * Copyright (C) 1998-2010 Ales Hvezda
-  * Copyright (C) 2007-2010 Peter Clifton
-+ * Copyright (C) 2011-2015 gEDA Contributors
-+ * Copyright (C) 2017-2019 Lepton EDA Contributors
-  *
-  * This library is free software; you can redistribute it and/or
-  * modify it under the terms of the GNU Library General Public
-@@ -56,6 +57,9 @@ void geda_list_add_glist( GedaList *list
- void geda_list_remove( GedaList *list, gpointer item );
- /*void geda_list_remove_glist( GedaList *list, GList *items ); */ /* Undemanded as yet */
- void geda_list_remove_all( GedaList *list );
-+void geda_list_move_item( GedaList* list, gpointer item, gint newpos );
-+
-+
- 
- /*const GList *geda_list_get_glist( GedaList *list ); */
- #define geda_list_get_glist(list) (list->glist)

Property changes on: cad/lepton-eda/files/patch-liblepton_include_liblepton_geda__list.h
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-liblepton_scheme_geda_log-rotate.scm
===================================================================
--- cad/lepton-eda/files/patch-liblepton_scheme_geda_log-rotate.scm	(revision 526543)
+++ cad/lepton-eda/files/patch-liblepton_scheme_geda_log-rotate.scm	(nonexistent)
@@ -1,11 +0,0 @@
---- liblepton/scheme/geda/log-rotate.scm.orig	2019-10-03 20:45:55 UTC
-+++ liblepton/scheme/geda/log-rotate.scm
-@@ -44,7 +44,7 @@
- ;; FIXME This should be obtained from the code that decides where to
- ;; create and populate log files.
- (define (log-directory)
--  (string-append (user-config-dir) file-name-separator-string "logs"))
-+  (string-append (user-cache-dir) file-name-separator-string "logs"))
- 
- ;;;; get-log-toolname path
- ;;

Property changes on: cad/lepton-eda/files/patch-liblepton_scheme_geda_log-rotate.scm
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-liblepton_scheme_geda_os.scm
===================================================================
--- cad/lepton-eda/files/patch-liblepton_scheme_geda_os.scm	(revision 526543)
+++ cad/lepton-eda/files/patch-liblepton_scheme_geda_os.scm	(nonexistent)
@@ -1,10 +0,0 @@
---- liblepton/scheme/geda/os.scm.orig	2019-10-03 20:45:55 UTC
-+++ liblepton/scheme/geda/os.scm
-@@ -49,6 +49,7 @@
- (define-public sys-config-dirs %sys-config-dirs)
- (define-public user-data-dir %user-data-dir)
- (define-public user-config-dir %user-config-dir)
-+(define-public user-cache-dir %user-cache-dir)
- 
- (define-public expand-env-variables
-   ;; Only compile regular expression once

Property changes on: cad/lepton-eda/files/patch-liblepton_scheme_geda_os.scm
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-liblepton_src_geda__list.c
===================================================================
--- cad/lepton-eda/files/patch-liblepton_src_geda__list.c	(revision 526543)
+++ cad/lepton-eda/files/patch-liblepton_src_geda__list.c	(nonexistent)
@@ -1,36 +0,0 @@
---- liblepton/src/geda_list.c.orig	2019-10-03 20:45:55 UTC
-+++ liblepton/src/geda_list.c
-@@ -1,7 +1,8 @@
--/* gEDA - GPL Electronic Design Automation
-- * libgeda - gEDA's library
-+/* Lepton EDA library
-  * Copyright (C) 1998-2000 Ales Hvezda
-  * Copyright (C) 2007-2010 Peter Clifton
-+ * Copyright (C) 2011-2013 gEDA Contributors
-+ * Copyright (C) 2017-2019 Lepton EDA Contributors
-  *
-  * This program is free software; you can redistribute it and/or modify
-  * it under the terms of the GNU General Public License as published by
-@@ -179,3 +180,22 @@ void geda_list_remove_all( GedaList *lis
-   g_signal_emit( list, geda_list_signals[ CHANGED ], 0 );
- }
- 
-+
-+/*! \brief Moves the list data \a item to a new position \a newpos.
-+ */
-+void geda_list_move_item( GedaList* list, gpointer item, gint newpos )
-+{
-+  GList* gl = list->glist;
-+  GList* node = g_list_find (gl, item);
-+
-+  if (node != NULL)
-+  {
-+    gl = g_list_remove_link (gl, node);
-+    gl = g_list_insert (gl, item, newpos);
-+    g_list_free (node);
-+    list->glist = gl;
-+
-+    g_signal_emit( list, geda_list_signals[ CHANGED ], 0 );
-+  }
-+}
-+

Property changes on: cad/lepton-eda/files/patch-liblepton_src_geda__list.c
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-liblepton_src_scheme__os.c
===================================================================
--- cad/lepton-eda/files/patch-liblepton_src_scheme__os.c	(revision 526543)
+++ cad/lepton-eda/files/patch-liblepton_src_scheme__os.c	(nonexistent)
@@ -1,43 +0,0 @@
---- liblepton/src/scheme_os.c.orig	2019-10-03 20:45:55 UTC
-+++ liblepton/src/scheme_os.c
-@@ -153,6 +153,25 @@ SCM_DEFINE (user_config_dir, "%user-conf
-   return scm_from_locale_string(eda_get_user_config_dir());
- }
- 
-+/*! \brief Get user cache directory.
-+ * \par Function Description
-+ * Returns the directory where per-user cache data should be
-+ * stored
-+ *
-+ * \note Scheme API: Implements the %user-cache-dir procedure is the
-+ * (geda core os) module.
-+ *
-+ * \return a string.
-+ */
-+SCM_DEFINE (user_cache_dir, "%user-cache-dir", 0, 0, 0, (),
-+            "Return the directory for user cache data.")
-+{
-+  /* eda_get_user_cache_dir() returns a raw environment string, so assume
-+   * it's in the current locale's encoding. */
-+  return scm_from_locale_string (eda_get_user_cache_dir ());
-+}
-+
-+
- /*!
-  * \brief Create the (geda core os) Scheme module.
-  * \par Function Description
-@@ -166,8 +185,12 @@ init_module_geda_core_os (void *unused)
-   #include "scheme_os.x"
- 
-   /* Add them to the module's public definitions. */
--  scm_c_export (s_platform, s_sys_data_dirs, s_sys_config_dirs,
--                s_user_data_dir, s_user_config_dir,
-+  scm_c_export (s_platform,
-+                s_sys_config_dirs,
-+                s_sys_data_dirs,
-+                s_user_cache_dir,
-+                s_user_config_dir,
-+                s_user_data_dir,
-                 NULL);
- }
- 

Property changes on: cad/lepton-eda/files/patch-liblepton_src_scheme__os.c
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-netlist_scheme_backend_gnet-spice-sdb.scm
===================================================================
--- cad/lepton-eda/files/patch-netlist_scheme_backend_gnet-spice-sdb.scm	(revision 526543)
+++ cad/lepton-eda/files/patch-netlist_scheme_backend_gnet-spice-sdb.scm	(nonexistent)
@@ -1,11 +0,0 @@
---- netlist/scheme/backend/gnet-spice-sdb.scm.orig	2019-10-03 20:45:55 UTC
-+++ netlist/scheme/backend/gnet-spice-sdb.scm
-@@ -209,7 +209,7 @@
-     (and (string-ci=? (gnetlist:get-package-attribute package "device")
-                       "spice-io")
-          package))
--  (filter-map spice-io? ls))
-+  (filter-map spice-io? package-list))
- 
- 
- ;;; Given a list of spice-IO packages (refdeses), this function

Property changes on: cad/lepton-eda/files/patch-netlist_scheme_backend_gnet-spice-sdb.scm
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-schematic_src_x__tabs.c
===================================================================
--- cad/lepton-eda/files/patch-schematic_src_x__tabs.c	(revision 526543)
+++ cad/lepton-eda/files/patch-schematic_src_x__tabs.c	(nonexistent)
@@ -1,275 +0,0 @@
---- schematic/src/x_tabs.c.orig	2019-10-03 20:45:55 UTC
-+++ schematic/src/x_tabs.c
-@@ -53,7 +53,7 @@
-  * key:         use-tabs
-  * group:       schematic.gui
-  * type:        boolean
-- * default val: false
-+ * default val: true
-  *
-  * 2) Whether to show "close" button on tabs:
-  * key:         show-close-button
-@@ -75,7 +75,7 @@
- 
- 
- static gboolean
--g_x_tabs_enabled = FALSE;
-+g_x_tabs_enabled = TRUE;
- 
- static gboolean
- g_x_tabs_show_close_button = TRUE;
-@@ -284,6 +284,28 @@ x_tabs_page_on_sel (GtkNotebook* nbook,
-                     guint        ndx,
-                     gpointer     data);
- 
-+static void
-+x_tabs_page_on_reordered (GtkNotebook* nbook,
-+                          GtkWidget*   wtab,
-+                          guint        newindex,
-+                          gpointer     data);
-+
-+
-+static gboolean
-+x_tabs_hdr_on_mouse_click (GtkWidget* hdr, GdkEvent* e, gpointer data);
-+static GtkMenu*
-+x_tabs_menu_create (TabInfo* nfo);
-+static void
-+x_tabs_menu_create_item (GschemToplevel* toplevel,
-+                         GtkWidget*      menu,
-+                         const gchar*    action_name,
-+                         const gchar*    action_label,
-+                         const gchar*    icon_name);
-+static void
-+x_tabs_menu_create_item_separ (GtkWidget* menu);
-+static void
-+x_tabs_menu_item_on_activate (GtkAction* action, gpointer data);
-+
- 
- 
- /* page view: */
-@@ -409,7 +431,24 @@ x_tabs_dbg_pages_dump (GschemToplevel* w
-   printf( " ^^^^^^^^^^^^^^ pages ^^^^^^^^^^^^^^^^^^\n\n" );
- }
- 
--#endif
-+static void
-+x_tabs_dbg_pages_dump_simple (GschemToplevel* w_current)
-+{
-+  printf( " >> pages:\n" );
-+  g_return_if_fail( w_current != NULL );
-+
-+  for ( GList* node = w_current->toplevel->pages->glist;
-+        node != NULL;
-+        node = g_list_next( node ) )
-+  {
-+    PAGE* p = node->data;
-+    printf( "    p: [%s]\n", g_path_get_basename( s_page_get_filename(p) ) );
-+  }
-+
-+  printf( "\n" );
-+}
-+
-+#endif /* DEBUG */
- 
- 
- 
-@@ -661,6 +700,11 @@ x_tabs_nbook_create (GschemToplevel* w_c
-   g_signal_connect (nbook, "switch-page",
-                     G_CALLBACK (&x_tabs_page_on_sel), w_current);
- 
-+  g_signal_connect (nbook,
-+                    "page-reordered",
-+                    G_CALLBACK (&x_tabs_page_on_reordered),
-+                    w_current);
-+
- } /* x_tabs_nbook_create() */
- 
- 
-@@ -913,7 +957,18 @@ x_tabs_hdr_set (GtkNotebook* nbook, TabI
-   g_return_if_fail (nfo != NULL);
- 
-   GtkWidget* hdr = x_tabs_hdr_create (nfo);
--  gtk_notebook_set_tab_label (nbook, nfo->wtab_, hdr);
-+
-+  GtkWidget* ebox = gtk_event_box_new();
-+  gtk_event_box_set_visible_window (GTK_EVENT_BOX (ebox), FALSE);
-+  gtk_container_add (GTK_CONTAINER (ebox), hdr);
-+  gtk_widget_show_all (ebox);
-+
-+  g_signal_connect (ebox,
-+                    "button-press-event",
-+                    G_CALLBACK (&x_tabs_hdr_on_mouse_click),
-+                    nfo);
-+
-+  gtk_notebook_set_tab_label (nbook, nfo->wtab_, ebox);
- }
- 
- 
-@@ -1160,6 +1215,8 @@ x_tabs_page_new (GschemToplevel* w_curre
-   x_tabs_tl_pview_cur_set (w_current, pview);
-   gint ndx = x_tabs_nbook_page_add (w_current, page, pview, wtab);
- 
-+  gtk_notebook_set_tab_reorderable (w_current->xtabs_nbook, wtab, TRUE);
-+
-   return x_tabs_info_add (w_current, ndx, page, pview, wtab);
- 
- } /* x_tabs_page_new() */
-@@ -1485,3 +1542,158 @@ x_tabs_page_on_sel (GtkNotebook* nbook,
- 
- } /* x_tabs_page_on_sel() */
- 
-+
-+
-+/*! \brief GtkNotebook "page-reordered" signal handler.
-+ */
-+static void
-+x_tabs_page_on_reordered (GtkNotebook* nbook,
-+                          GtkWidget*   wtab,
-+                          guint        newindex,
-+                          gpointer     data)
-+{
-+  GschemToplevel* w_current = (GschemToplevel*) data;
-+  g_return_if_fail (w_current != NULL);
-+  g_return_if_fail (w_current->toplevel != NULL);
-+  g_return_if_fail (w_current->toplevel->pages != NULL);
-+
-+  TabInfo* nfo = x_tabs_info_find_by_wtab (w_current->xtabs_info_list, wtab);
-+  g_return_if_fail (nfo != NULL);
-+
-+  GedaPageList* pages = w_current->toplevel->pages;
-+  geda_list_move_item (pages, nfo->page_, newindex);
-+
-+  gtk_widget_grab_focus (GTK_WIDGET (nfo->pview_));
-+  page_select_widget_update (w_current);
-+
-+#ifdef DEBUG
-+  x_tabs_dbg_pages_dump_simple( w_current );
-+#endif
-+
-+} /* x_tabs_page_on_reordered() */
-+
-+
-+
-+/*! \brief Create popup menu for tab's header.
-+ */
-+static GtkMenu*
-+x_tabs_menu_create (TabInfo* nfo)
-+{
-+  g_return_val_if_fail (nfo != NULL, NULL);
-+
-+  GschemToplevel* tl = nfo->tl_;
-+  g_return_val_if_fail (tl != NULL, NULL);
-+
-+  GtkWidget* menu = gtk_menu_new();
-+  x_tabs_menu_create_item (tl, menu, "file-new", _("_New"), GTK_STOCK_NEW);
-+  x_tabs_menu_create_item (tl, menu, "file-open", _("_Open"), GTK_STOCK_OPEN);
-+  x_tabs_menu_create_item_separ (menu);
-+  x_tabs_menu_create_item (tl, menu, "file-save", _("_Save"), GTK_STOCK_SAVE);
-+  x_tabs_menu_create_item (tl, menu, "file-save-as", _("Save _As..."), GTK_STOCK_SAVE_AS);
-+  x_tabs_menu_create_item_separ (menu);
-+  x_tabs_menu_create_item (tl, menu, "page-manager", _("Page _Manager..."), NULL);
-+  x_tabs_menu_create_item_separ (menu);
-+  x_tabs_menu_create_item (tl, menu, "page-close", _("_Close"), GTK_STOCK_CLOSE);
-+
-+  gtk_widget_show_all (menu);
-+  return GTK_MENU (menu);
-+
-+} /* x_tabs_menu_create() */
-+
-+
-+
-+/*! \brief Tab's header widget "button-press-event" signal handler.
-+ *  \todo  Consider switching to clicked tab
-+ */
-+static gboolean
-+x_tabs_hdr_on_mouse_click (GtkWidget* hdr, GdkEvent* e, gpointer data)
-+{
-+  g_return_val_if_fail (data != NULL, FALSE);
-+  GdkEventButton* ebtn = (GdkEventButton*) e;
-+
-+  TabInfo* nfo    = (TabInfo*) data;
-+  TabInfo* nfocur = x_tabs_info_cur (nfo->tl_);
-+
-+  /* show menu for current tab only:
-+  */
-+  if (nfo != nfocur)
-+    return FALSE;
-+
-+#ifdef DEBUG
-+  printf( "p: [%s]\n",   g_path_get_basename( s_page_get_filename(nfo->page_) ) );
-+  printf( "C: [%s]\n\n", g_path_get_basename( s_page_get_filename(nfocur->page_) ) );
-+#endif
-+
-+  if (ebtn->type == GDK_BUTTON_PRESS && ebtn->button == 3)
-+  {
-+    GtkMenu* menu = x_tabs_menu_create (nfo);
-+
-+    int btn = 0;
-+    int etime = 0;
-+    if (ebtn != NULL)
-+    {
-+      btn = ebtn->button;
-+      etime = gtk_get_current_event_time();
-+    }
-+
-+    gtk_menu_attach_to_widget (menu, hdr, NULL);
-+    gtk_menu_popup (menu, NULL, NULL, NULL, NULL, btn, etime);
-+
-+    return TRUE;
-+  }
-+
-+  return FALSE;
-+
-+} /* x_tabs_page_on_mouse_click() */
-+
-+
-+
-+/*! \brief "activate" signal handler for context menu item action.
-+ */
-+static void
-+x_tabs_menu_item_on_activate (GtkAction* action, gpointer data)
-+{
-+  GschemToplevel* toplevel    = (GschemToplevel*) data;
-+  const gchar*    action_name = gtk_action_get_name (action);
-+
-+  g_action_eval_by_name (toplevel, action_name);
-+}
-+
-+
-+
-+/*! \brief Create and add popup menu item separator.
-+ */
-+static void
-+x_tabs_menu_create_item_separ (GtkWidget* menu)
-+{
-+  gtk_menu_shell_append (GTK_MENU_SHELL (menu),
-+                         gtk_separator_menu_item_new());
-+}
-+
-+
-+
-+/*! \brief Create and add popup menu item.
-+ */
-+static void
-+x_tabs_menu_create_item (GschemToplevel* toplevel,
-+                         GtkWidget*      menu,
-+                         const gchar*    action_name,
-+                         const gchar*    action_label,
-+                         const gchar*    icon_name)
-+{
-+  GschemAction* action = gschem_action_new (action_name,  /* name */
-+                                            action_label, /* label */
-+                                            NULL,         /* tooltip */
-+                                            icon_name,    /* stock_id */
-+                                            NULL);        /* multikey_accel */
-+
-+  GtkWidget* item = gtk_action_create_menu_item (GTK_ACTION (action));
-+  gtk_menu_shell_append (GTK_MENU_SHELL (menu), item);
-+
-+  g_signal_connect (action,
-+                    "activate",
-+                    G_CALLBACK (&x_tabs_menu_item_on_activate),
-+                    toplevel);
-+
-+} /* x_tabs_menu_create_item() */
-+

Property changes on: cad/lepton-eda/files/patch-schematic_src_x__tabs.c
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-utils_scripts_lepton-upcfg.in
===================================================================
--- cad/lepton-eda/files/patch-utils_scripts_lepton-upcfg.in	(nonexistent)
+++ cad/lepton-eda/files/patch-utils_scripts_lepton-upcfg.in	(working copy)
@@ -0,0 +1,17 @@
+--- utils/scripts/lepton-upcfg.in.orig	2020-03-25 12:47:33 UTC
++++ utils/scripts/lepton-upcfg.in
+@@ -219,7 +219,13 @@ Lepton EDA homepage: <~a>
+   )
+ 
+ 
+-  ( upcfg-log "ii: upgrading config in [~a]...~%" (config-file-path cfg-id) )
++  ( upcfg-log
++    "ii: upgrading config in [~a]...~%"
++    ( if cfg-id
++      ( config-file-path cfg-id ) ; if
++      ( list-ref files 0 )        ; else
++    )
++  )
+ 
+   ( if ( null? files )
+     ( or (config-upgrade cfg-id #:copy copy #:overwrite overwrite) (failure) ) ; if

Property changes on: cad/lepton-eda/files/patch-utils_scripts_lepton-upcfg.in
___________________________________________________________________
Added: fbsd:nokeywords
## -0,0 +1 ##
+yes
\ No newline at end of property
Added: svn:eol-style
## -0,0 +1 ##
+native
\ No newline at end of property
Added: svn:mime-type
## -0,0 +1 ##
+text/plain
\ No newline at end of property
Index: cad/lepton-eda/files/patch-version.h
===================================================================
--- cad/lepton-eda/files/patch-version.h	(revision 526543)
+++ cad/lepton-eda/files/patch-version.h	(working copy)
@@ -1,11 +1,11 @@
---- version.h.orig	2019-10-04 15:45:44 UTC
+--- version.h.orig	2020-03-25 05:39:40 UTC
 +++ version.h
 @@ -0,0 +1,8 @@
 +/* Date part of package version string. */
-+#define PACKAGE_DATE_VERSION "20191003"
++#define PACKAGE_DATE_VERSION "20200319"
 +
 +/* Dotted part of package version string. */
-+#define PACKAGE_DOTTED_VERSION "1.9.9"
++#define PACKAGE_DOTTED_VERSION "1.9.10"
 +
 +/* Current git commit. */
-+#define PACKAGE_GIT_COMMIT "3139e7782564ff4bf76963984a735fbe638e43a0"
++#define PACKAGE_GIT_COMMIT "c69155b34b82d2b288f4787f92fe930112e73c72"
Index: cad/lepton-eda/pkg-descr
===================================================================
--- cad/lepton-eda/pkg-descr	(revision 526543)
+++ cad/lepton-eda/pkg-descr	(working copy)
@@ -1,6 +1,8 @@
-The Lepton EDA suite provides schematic capture, netlisting, bill
-of materials generation, and many other features.  It was forked
-from the gEDA/gaf suite in late 2016.  It's in active development
-and well supported.
+Lepton EDA is a suite of free software tools for designing
+electronics. It provides schematic capture, netlisting into
+over 30 netlist formats, and many other features.
+It was forked from the gEDA/gaf suite in late 2016 by most
+of its active developers at that time.
+It's in active development and well supported.
 
 WWW: https://github.com/lepton-eda/lepton-eda
Index: cad/lepton-eda/pkg-plist
===================================================================
--- cad/lepton-eda/pkg-plist	(revision 526543)
+++ cad/lepton-eda/pkg-plist	(working copy)
@@ -1,6 +1,7 @@
 bin/lepton-archive
 bin/lepton-attrib
 bin/lepton-cli
+bin/lepton-embed
 bin/lepton-netlist
 bin/lepton-pcb_backannotate
 bin/lepton-refdes_renum
@@ -8,7 +9,6 @@
 bin/lepton-sch2pcb
 bin/lepton-schdiff
 bin/lepton-schematic
-bin/lepton-schlas
 bin/lepton-symcheck
 bin/lepton-symfix
 bin/lepton-tragesym
@@ -32,8 +32,8 @@
 include/liblepton/geda_circle_object.h
 include/liblepton/geda_color.h
 include/liblepton/geda_color_map.h
-include/liblepton/geda_complex.h
-include/liblepton/geda_complex_object.h
+include/liblepton/geda_component.h
+include/liblepton/geda_component_object.h
 include/liblepton/geda_coord.h
 include/liblepton/geda_fill_type.h
 include/liblepton/geda_forward.h
@@ -68,16 +68,17 @@
 include/libleptonrenderer/edarenderer.h
 include/libleptonrenderer/libleptonrenderer.h
 lib/liblepton.so
-lib/liblepton.so.4
-lib/liblepton.so.4.0.0
+lib/liblepton.so.5
+lib/liblepton.so.5.0.0
 lib/libleptonrenderer.so
-lib/libleptonrenderer.so.1
-lib/libleptonrenderer.so.1.0.2
+lib/libleptonrenderer.so.2
+lib/libleptonrenderer.so.2.0.0
 libdata/pkgconfig/liblepton.pc
 libdata/pkgconfig/libleptonrenderer.pc
 man/man1/lepton-archive.1.gz
 man/man1/lepton-attrib.1.gz
 man/man1/lepton-cli.1.gz
+man/man1/lepton-embed.1.gz
 man/man1/lepton-netlist.1.gz
 man/man1/lepton-pcb_backannotate.1.gz
 man/man1/lepton-refdes_renum.1.gz
@@ -85,14 +86,14 @@
 man/man1/lepton-sch2pcb.1.gz
 man/man1/lepton-schdiff.1.gz
 man/man1/lepton-schematic.1.gz
-man/man1/lepton-schlas.1.gz
 man/man1/lepton-symcheck.1.gz
 man/man1/lepton-symfix.1.gz
 man/man1/lepton-tragesym.1.gz
 share/applications/lepton-attrib.desktop
 share/applications/lepton-schematic.desktop
+%%PORTDOCS%%%%DOCSDIR%%/NEWS.md
+%%PORTDOCS%%%%DOCSDIR%%/README.md
 %%PORTDOCS%%%%DOCSDIR%%/gedadocs.html
-%%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/lepton-scheme.css
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Actions.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Arcs.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Attributes.html
@@ -112,15 +113,11 @@
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Core-object-functions.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Core-page-functions.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Core-source-library-functions.html
-%%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Files-and-pages.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Function-Index.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/General-object-functions.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Hooks.html
-%%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/index.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Introduction.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Key-mapping.html
-%%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/lepton_002dnetlist-API-Reference.html
-%%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/lepton_002dschematic-API-Reference.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Lines.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Logging-functions.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Miscellanous-lepton_002dschematic-functions.html
@@ -128,9 +125,9 @@
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Object-bounds.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Object-color.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Object-fill-and-stroke.html
-%%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Objects.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Object-sub_002dtypes.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Object-transformations.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Objects.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Pages.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Paths.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Pictures.html
@@ -141,9 +138,14 @@
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Text.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Variable-Index.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Windows-and-views.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/index.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/lepton-scheme.css
+%%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/lepton_002dnetlist-API-Reference.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/lepton_002dschematic-API-Reference.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-archive.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-attrib.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-cli.html
+%%PORTDOCS%%%%DOCSDIR%%/man/lepton-embed.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-netlist.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-pcb_backannotate.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-refdes_renum.html
@@ -151,7 +153,6 @@
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-sch2pcb.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-schdiff.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-schematic.html
-%%PORTDOCS%%%%DOCSDIR%%/man/lepton-schlas.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-symcheck.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-symfix.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-tragesym.html
@@ -623,6 +624,7 @@
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/Q2.cir
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/Q2.sch
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/README
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/Simulation.batch.cmd
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/Simulation.cmd
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/gafrc
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/model/BJTM1_Q1.mod
@@ -635,42 +637,75 @@
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/model/DiodeM3_Q2.mod
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/sym/BJT_Model.sym
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/sym/Q_Model.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/sym/capacitor-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/sym/diode-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/sym/gnd-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/sym/inductor-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/sym/resistor-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/sym/spice-include-1.sym
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/sym/spice-subcircuit-IO-1.sym
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/sym/spice-subcircuit-LL-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/sym/title-B.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/sym/vac-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/sym/vdc-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/Simulation.batch.cmd
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/Simulation.cmd
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/TwoStageAmp.sch
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/gafrc
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/models/2N3904.mod
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/spice.netlist
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/sym/capacitor-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/sym/gnd-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/sym/resistor-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/sym/spice-directive-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/sym/spice-include-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/sym/spice-model-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/sym/title-B.sym
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/sym/transistor.sym
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/7414-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/sym/vcc-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/sym/vdc-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/TwoStageAmp/sym/vsin-1.sym
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/README
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/attribs
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/copyleft.sym
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/crdist.sh
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/cy7c64603-52nc.sym
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG-consio.ps
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG-consio.sch
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG-consio.sym
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG-jtagio.ps
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG-jtagio.sch
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG-jtagio.sym
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG-pcb.net
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG-psu.ps
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG-psu.sch
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG-psu.sym
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG-ucont.ps
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG-ucont.sch
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG-ucont.sym
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG.bom
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG.ps
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gTAG.sch
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gafrc
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/gnetlistrc
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/max882.sym
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sn75240pw.sym
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/bom
-%%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/bom.readme
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/geda.conf
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/ps/gTAG-consio.ps
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/ps/gTAG-jtagio.ps
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/ps/gTAG-psu.ps
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/ps/gTAG-ucont.ps
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/ps/gTAG.ps
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/24Cxx-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/3.3V-plus-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/5V-plus-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/7414-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/capacitor-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/capacitor-2.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/connector4-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/connector8-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/copyleft.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/crystal-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/cy7c64603-52nc.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/gTAG-consio.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/gTAG-jtagio.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/gTAG-psu.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/gTAG-ucont.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/gnd-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/header20-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/in-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/jumper-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/max882.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/out-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/resistor-2.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/sn75240pw.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/gTAG/sym/title-A2.sym
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/gafrc
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/lightning.net
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/lightning.pcb
@@ -678,6 +713,14 @@
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/lightning.sch.bom
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/sym/2N4401.sym
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/sym/2N4403.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/sym/capacitor-2.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/sym/diode-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/sym/in-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/sym/inductor-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/sym/out-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/sym/resistor-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/sym/resistor-variable-1.sym
+%%PORTEXAMPLES%%%%EXAMPLESDIR%%/lightning_detector/sym/title-A4.sym
 share/icons/hicolor/16x16/mimetypes/application-x-lepton-sch2pcb-project.png
 share/icons/hicolor/16x16/mimetypes/application-x-lepton-schematic.png
 share/icons/hicolor/16x16/mimetypes/application-x-lepton-symbol.png
@@ -738,13 +781,10 @@
 %%DATADIR%%/bitmap/gschem-top2bottom.png
 %%DATADIR%%/bitmap/gschem-undo.xpm
 %%DATADIR%%/bitmap/gschem-warning.png
-%%DATADIR%%/gafrc.d/config-netlist.scm
-%%DATADIR%%/gafrc.d/geda-clib.scm
 %%DATADIR%%/geda-system.conf
 %%DATADIR%%/gschem-colormap-bw
 %%DATADIR%%/gschem-colormap-darkbg
 %%DATADIR%%/gschem-colormap-lightbg
-%%DATADIR%%/gschem-gtkrc
 %%DATADIR%%/icons/hicolor/16x16/actions/attribute-attach.png
 %%DATADIR%%/icons/hicolor/16x16/actions/attribute-detach.png
 %%DATADIR%%/icons/hicolor/16x16/actions/attribute-show-both.png
@@ -771,6 +811,7 @@
 %%DATADIR%%/icons/hicolor/22x22/actions/insert-symbol.png
 %%DATADIR%%/icons/hicolor/22x22/actions/insert-text.png
 %%DATADIR%%/icons/hicolor/22x22/actions/select.png
+%%DATADIR%%/lepton-gtkrc
 %%DATADIR%%/lepton-system.conf
 %%DATADIR%%/print-colormap-darkbg
 %%DATADIR%%/print-colormap-lightbg
@@ -778,6 +819,8 @@
 %%DATADIR%%/scheme/auto-place-netname.scm
 %%DATADIR%%/scheme/auto-refdes.scm
 %%DATADIR%%/scheme/auto-uref.scm
+%%DATADIR%%/scheme/autoload/config-netlist.scm
+%%DATADIR%%/scheme/autoload/config-symbol-libraries.scm
 %%DATADIR%%/scheme/backend/gnet-PCB.scm
 %%DATADIR%%/scheme/backend/gnet-allegro.scm
 %%DATADIR%%/scheme/backend/gnet-bae.scm
@@ -821,7 +864,6 @@
 %%DATADIR%%/scheme/conf/schematic/deprecated.scm
 %%DATADIR%%/scheme/conf/schematic/keys.scm
 %%DATADIR%%/scheme/conf/schematic/menu.scm
-%%DATADIR%%/scheme/conf/schematic/options.scm
 %%DATADIR%%/scheme/conf/schematic/stroke.scm
 %%DATADIR%%/scheme/default-attrib-positions.scm
 %%DATADIR%%/scheme/geda-deprecated-config.scm
@@ -828,69 +870,83 @@
 %%DATADIR%%/scheme/geda.scm
 %%DATADIR%%/scheme/geda/attrib.scm
 %%DATADIR%%/scheme/geda/config.scm
-%%DATADIR%%/scheme/geda/core/gettext.scm
 %%DATADIR%%/scheme/geda/deprecated.scm
 %%DATADIR%%/scheme/geda/library.scm
-%%DATADIR%%/scheme/geda/log-rotate.scm
 %%DATADIR%%/scheme/geda/log.scm
 %%DATADIR%%/scheme/geda/object.scm
 %%DATADIR%%/scheme/geda/os.scm
 %%DATADIR%%/scheme/geda/page.scm
-%%DATADIR%%/scheme/geda/repl.scm
 %%DATADIR%%/scheme/gschem/action.scm
 %%DATADIR%%/scheme/gschem/attrib.scm
 %%DATADIR%%/scheme/gschem/builtins.scm
-%%DATADIR%%/scheme/gschem/core/gettext.scm
 %%DATADIR%%/scheme/gschem/deprecated.scm
 %%DATADIR%%/scheme/gschem/gschemdoc.scm
 %%DATADIR%%/scheme/gschem/hook.scm
 %%DATADIR%%/scheme/gschem/keymap.scm
-%%DATADIR%%/scheme/gschem/repl.scm
 %%DATADIR%%/scheme/gschem/selection.scm
 %%DATADIR%%/scheme/gschem/symbol/check.scm
 %%DATADIR%%/scheme/gschem/util.scm
 %%DATADIR%%/scheme/gschem/window.scm
+%%DATADIR%%/scheme/lepton/attrib.scm
+%%DATADIR%%/scheme/lepton/config.scm
+%%DATADIR%%/scheme/lepton/core/gettext.scm
 %%DATADIR%%/scheme/lepton/file-system.scm
 %%DATADIR%%/scheme/lepton/legacy-config.scm
 %%DATADIR%%/scheme/lepton/legacy-config/keylist.scm
 %%DATADIR%%/scheme/lepton/library.scm
 %%DATADIR%%/scheme/lepton/library/component.scm
+%%DATADIR%%/scheme/lepton/log-rotate.scm
+%%DATADIR%%/scheme/lepton/log.scm
+%%DATADIR%%/scheme/lepton/object.scm
+%%DATADIR%%/scheme/lepton/os.scm
+%%DATADIR%%/scheme/lepton/page.scm
 %%DATADIR%%/scheme/lepton/rc.scm
+%%DATADIR%%/scheme/lepton/repl.scm
 %%DATADIR%%/scheme/lepton/version.scm
-%%DATADIR%%/scheme/list-keys.scm
 %%DATADIR%%/scheme/netlist.scm
 %%DATADIR%%/scheme/netlist/attrib/compare.scm
+%%DATADIR%%/scheme/netlist/attrib/refdes.scm
 %%DATADIR%%/scheme/netlist/backend-getopt.scm
 %%DATADIR%%/scheme/netlist/config.scm
 %%DATADIR%%/scheme/netlist/core/gettext.scm
 %%DATADIR%%/scheme/netlist/deprecated.scm
+%%DATADIR%%/scheme/netlist/duplicate.scm
 %%DATADIR%%/scheme/netlist/error.scm
 %%DATADIR%%/scheme/netlist/hierarchy.scm
+%%DATADIR%%/scheme/netlist/mode.scm
 %%DATADIR%%/scheme/netlist/net.scm
 %%DATADIR%%/scheme/netlist/option.scm
 %%DATADIR%%/scheme/netlist/package-pin.scm
 %%DATADIR%%/scheme/netlist/package.scm
-%%DATADIR%%/scheme/netlist/page.scm
 %%DATADIR%%/scheme/netlist/partlist.scm
 %%DATADIR%%/scheme/netlist/partlist/common.scm
-%%DATADIR%%/scheme/netlist/pin-net.scm
 %%DATADIR%%/scheme/netlist/port.scm
-%%DATADIR%%/scheme/netlist/rename.scm
-%%DATADIR%%/scheme/netlist/repl.scm
 %%DATADIR%%/scheme/netlist/schematic-component.scm
 %%DATADIR%%/scheme/netlist/schematic-connection.scm
+%%DATADIR%%/scheme/netlist/schematic-port.scm
 %%DATADIR%%/scheme/netlist/schematic.scm
 %%DATADIR%%/scheme/netlist/schematic/toplevel.scm
-%%DATADIR%%/scheme/netlist/sort.scm
-%%DATADIR%%/scheme/netlist/traverse.scm
+%%DATADIR%%/scheme/netlist/subschematic-connection.scm
+%%DATADIR%%/scheme/netlist/subschematic.scm
 %%DATADIR%%/scheme/netlist/verbose.scm
 %%DATADIR%%/scheme/pcb.scm
-%%DATADIR%%/scheme/print-NB-attribs.scm
+%%DATADIR%%/scheme/schematic/action.scm
+%%DATADIR%%/scheme/schematic/attrib.scm
+%%DATADIR%%/scheme/schematic/builtins.scm
+%%DATADIR%%/scheme/schematic/core/gettext.scm
+%%DATADIR%%/scheme/schematic/gschemdoc.scm
 %%DATADIR%%/scheme/schematic/gui/keymap.scm
 %%DATADIR%%/scheme/schematic/gui/stroke.scm
+%%DATADIR%%/scheme/schematic/hook.scm
+%%DATADIR%%/scheme/schematic/keymap.scm
 %%DATADIR%%/scheme/schematic/netlist.scm
 %%DATADIR%%/scheme/schematic/precompile.scm
+%%DATADIR%%/scheme/schematic/repl.scm
+%%DATADIR%%/scheme/schematic/selection.scm
+%%DATADIR%%/scheme/schematic/symbol/check.scm
 %%DATADIR%%/scheme/schematic/undo.scm
+%%DATADIR%%/scheme/schematic/util.scm
+%%DATADIR%%/scheme/schematic/window.scm
 %%DATADIR%%/scheme/symbol/blame.scm
 %%DATADIR%%/scheme/symbol/check.scm
 %%DATADIR%%/scheme/symbol/check/alignment.scm
