Index: cad/lepton-eda/Makefile
===================================================================
--- cad/lepton-eda/Makefile	(revision 557827)
+++ cad/lepton-eda/Makefile	(working copy)
@@ -1,9 +1,9 @@
 # $FreeBSD$
 
 PORTNAME=	lepton-eda
-PORTVERSION=	1.9.11
-DISTVERSIONSUFFIX=	-20200604
+PORTVERSION=	1.9.13
 CATEGORIES=	cad
+MASTER_SITES=	https://github.com/lepton-eda/lepton-eda/releases/download/${PORTVERSION}-20201211/
 
 MAINTAINER=	graahnul.grom@gmail.com
 COMMENT=	Lepton Electronic Design Automation
@@ -15,12 +15,12 @@
 		libgc-threaded.so:devel/boehm-gc-threaded \
 		libguile-2.2.so:lang/guile2 \
 		libpng.so:graphics/png \
-		libstroke.so:devel/libstroke
+		libstroke.so:devel/libstroke \
+		libgtkextra-x11-3.0.so:x11-toolkits/gtkextra
 
-USES=		autoreconf desktop-file-utils gettext-tools gnome \
+USES=		desktop-file-utils gettext-tools gnome \
 		groff:build libtool localbase makeinfo pathfix perl5 \
 		pkgconfig shared-mime-info shebangfix
-USE_GITHUB=	yes
 USE_GNOME=	cairo gtk20
 USE_LDCONFIG=	yes
 
@@ -35,7 +35,7 @@
 
 INSTALLS_ICONS=	yes
 
-INFO=	lepton-scheme
+INFO=	lepton-scheme lepton-manual
 
 OPTIONS_DEFINE=	DOCS EXAMPLES NLS
 OPTIONS_SUB=	yes
@@ -46,10 +46,10 @@
 post-patch:
 	${REINPLACE_CMD} -e \
 		's,\$$(docdir)/examples,\$$(datadir)/examples/lepton-eda,' \
-		${WRKSRC}/examples/Makefile.am \
-		${WRKSRC}/examples/RF_Amp/Makefile.am \
-		${WRKSRC}/examples/TwoStageAmp/Makefile.am \
-		${WRKSRC}/examples/gTAG/Makefile.am \
-		${WRKSRC}/examples/lightning_detector/Makefile.am
+		${WRKSRC}/examples/Makefile.in \
+		${WRKSRC}/examples/RF_Amp/Makefile.in \
+		${WRKSRC}/examples/TwoStageAmp/Makefile.in \
+		${WRKSRC}/examples/gTAG/Makefile.in \
+		${WRKSRC}/examples/lightning_detector/Makefile.in
 
 .include <bsd.port.mk>
Index: cad/lepton-eda/distinfo
===================================================================
--- cad/lepton-eda/distinfo	(revision 557827)
+++ cad/lepton-eda/distinfo	(working copy)
@@ -1,3 +1,3 @@
-TIMESTAMP = 1591284076
-SHA256 (lepton-eda-lepton-eda-1.9.11-20200604_GH0.tar.gz) = 40a3855aeaa00adc19a185bfca0f8959b259fce7ddfe3b36913efc9291983b53
-SIZE (lepton-eda-lepton-eda-1.9.11-20200604_GH0.tar.gz) = 8461913
+TIMESTAMP = 1607692128
+SHA256 (lepton-eda-1.9.13.tar.gz) = f417b71415a7654d8ce5a350c22dcdf0cf357b44bb9f8ea6c16421d66acae19f
+SIZE (lepton-eda-1.9.13.tar.gz) = 10155388
Index: cad/lepton-eda/files/patch-version.h
===================================================================
--- cad/lepton-eda/files/patch-version.h	(revision 557827)
+++ cad/lepton-eda/files/patch-version.h	(nonexistent)
@@ -1,11 +0,0 @@
---- version.h.orig	2020-06-04 15:22:30 UTC
-+++ version.h
-@@ -0,0 +1,8 @@
-+/* Date part of package version string. */
-+#define PACKAGE_DATE_VERSION "20200604"
-+
-+/* Dotted part of package version string. */
-+#define PACKAGE_DOTTED_VERSION "1.9.11"
-+
-+/* Current git commit. */
-+#define PACKAGE_GIT_COMMIT "219f66e870008a2bbc575b5b5f9499bf6c4a9b1f"

Property changes on: cad/lepton-eda/files/patch-version.h
___________________________________________________________________
Deleted: fbsd:nokeywords
## -1 +0,0 ##
-yes
\ No newline at end of property
Deleted: svn:eol-style
## -1 +0,0 ##
-native
\ No newline at end of property
Deleted: svn:mime-type
## -1 +0,0 ##
-text/plain
\ No newline at end of property
Index: cad/lepton-eda/pkg-plist
===================================================================
--- cad/lepton-eda/pkg-plist	(revision 557827)
+++ cad/lepton-eda/pkg-plist	(working copy)
@@ -14,9 +14,12 @@
 bin/lepton-tragesym
 bin/lepton-upcfg
 include/liblepton/defines.h
+include/liblepton/edacairo.h
 include/liblepton/edaconfig.h
 include/liblepton/edaerrors.h
+include/liblepton/edapangorenderer.h
 include/liblepton/edapaths.h
+include/liblepton/edarenderer.h
 include/liblepton/edascmhookproxy.h
 include/liblepton/edascmvaluetypes.h
 include/liblepton/funcs.h
@@ -31,7 +34,6 @@
 include/liblepton/geda_circle.h
 include/liblepton/geda_circle_object.h
 include/liblepton/geda_color.h
-include/liblepton/geda_color_map.h
 include/liblepton/geda_component.h
 include/liblepton/geda_component_object.h
 include/liblepton/geda_coord.h
@@ -65,21 +67,18 @@
 include/liblepton/prototype.h
 include/liblepton/struct.h
 include/libleptonattrib/prototype.h
-include/libleptonrenderer/edacairo.h
-include/libleptonrenderer/edarenderer.h
-include/libleptonrenderer/libleptonrenderer.h
 lib/liblepton.so
-lib/liblepton.so.6
-lib/liblepton.so.6.0.0
+lib/liblepton.so.7
+lib/liblepton.so.7.0.0
 lib/libleptonattrib.so
-lib/libleptonattrib.so.1
-lib/libleptonattrib.so.1.0.0
-lib/libleptonrenderer.so
-lib/libleptonrenderer.so.3
-lib/libleptonrenderer.so.3.0.0
+lib/libleptonattrib.so.2
+lib/libleptonattrib.so.2.0.0
+lib/libleptongui.so
+lib/libleptongui.so.1
+lib/libleptongui.so.1.0.0
 libdata/pkgconfig/liblepton.pc
 libdata/pkgconfig/libleptonattrib.pc
-libdata/pkgconfig/libleptonrenderer.pc
+libdata/pkgconfig/libleptongui.pc
 man/man1/lepton-archive.1.gz
 man/man1/lepton-attrib.1.gz
 man/man1/lepton-cli.1.gz
@@ -95,22 +94,18 @@
 man/man1/lepton-symfix.1.gz
 man/man1/lepton-tragesym.1.gz
 man/man1/lepton-upcfg.1.gz
+share/info/coordinate-space.png
+share/info/path-example.png
+share/info/text-layout.png
 share/applications/lepton-attrib.desktop
 share/applications/lepton-schematic.desktop
+share/mime/application/x-lepton-sch2pcb-project.xml
+share/mime/application/x-lepton-schematic.xml
+share/mime/application/x-lepton-symbol.xml
+share/mime/packages/liblepton.xml
 %%PORTDOCS%%%%DOCSDIR%%/NEWS.md
 %%PORTDOCS%%%%DOCSDIR%%/README.md
-%%PORTDOCS%%%%DOCSDIR%%/changelogs/changelog.docs
-%%PORTDOCS%%%%DOCSDIR%%/changelogs/changelog.examples
-%%PORTDOCS%%%%DOCSDIR%%/changelogs/changelog.examples.gTAG
-%%PORTDOCS%%%%DOCSDIR%%/changelogs/changelog.gattrib
-%%PORTDOCS%%%%DOCSDIR%%/changelogs/changelog.gnetlist
-%%PORTDOCS%%%%DOCSDIR%%/changelogs/changelog.gsch2pcb
-%%PORTDOCS%%%%DOCSDIR%%/changelogs/changelog.gschem
-%%PORTDOCS%%%%DOCSDIR%%/changelogs/changelog.gsymcheck
-%%PORTDOCS%%%%DOCSDIR%%/changelogs/changelog.libgeda
-%%PORTDOCS%%%%DOCSDIR%%/changelogs/changelog.symbols
-%%PORTDOCS%%%%DOCSDIR%%/changelogs/changelog.tragesym
-%%PORTDOCS%%%%DOCSDIR%%/changelogs/changelog.utils
+%%PORTDOCS%%%%DOCSDIR%%/nc.pdf
 %%PORTDOCS%%%%DOCSDIR%%/gedadocs.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Actions.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Arcs.html
@@ -155,11 +150,99 @@
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/System-information.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Text.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Variable-Index.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Version-information.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Windows-and-views.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/index.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/lepton-scheme.css
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/lepton_002dnetlist-API-Reference.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/lepton_002dschematic-API-Reference.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Attributes.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Colors.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Communication.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Compatibility-with-geda_002dgaf.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Component-libraries.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Component-library-setup.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Configuration-of-Lepton-tools.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Configuration.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Configure-options.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Coordinate-space.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Credits-and-history-of-lepton_002dattrib.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Debug-Glib-and-GTK.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Debug-Scheme-code.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Debuggers.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Debugging.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Dependencies.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Deprecated-settings.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Development.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Documentation.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Embedded-Components.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/File-format-overview.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Filenames.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Installation-from-git.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Installation-from-source.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Installation-of-lepton_002dattrib.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Installation.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Introduction.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Legacy-configuration.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/New-configuration-system.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Object-types.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Other-resources.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Overview.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Path-data.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Resources.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Topic-Index.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Troubleshooting-dependencies.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Usage-of-lepton_002dattrib.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Utilities.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Web-pages.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/arc.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/box.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/bus.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/circle.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component_002dlibrary.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component_002dlibrary_002dcommand.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component_002dlibrary_002dfuncs.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component_002dlibrary_002dsearch.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/coordinate-space.png
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/export-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/font.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gEDA-File-Format-Document-Revision-History.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gEDA-file-format.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gafrc.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gschemrc.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/index.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton-manual.css
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dattrib-menus.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dattrib.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dcli-configuration.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dnetlist-configuration.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dnetlist.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dschematic-configuration.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dschematic.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/line.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/netlist-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/netlist_002ehierarchy-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/path-example.png
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/path.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/picture.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/pin.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/reset_002dcomponent_002dlibrary.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002eattrib-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002ebackup-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002egui-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002elibrary-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002elog_002dwindow-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002emacro_002dwidget-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002eprinting-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002estatus_002dbar-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002etabs-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002eundo-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/text-and-attributes.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/text-layout.png
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/version.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-archive.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-attrib.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-cli.html
@@ -175,7 +258,6 @@
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-symfix.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-tragesym.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-upcfg.html
-%%PORTDOCS%%%%DOCSDIR%%/nc.pdf
 %%PORTDOCS%%%%DOCSDIR%%/readmes/README.bom
 %%PORTDOCS%%%%DOCSDIR%%/readmes/README.eagle
 %%PORTDOCS%%%%DOCSDIR%%/readmes/README.lepton-renum
@@ -800,10 +882,6 @@
 %%DATADIR%%/bitmap/gschem-top2bottom.png
 %%DATADIR%%/bitmap/gschem-undo.xpm
 %%DATADIR%%/bitmap/gschem-warning.png
-%%DATADIR%%/geda-system.conf
-%%DATADIR%%/gschem-colormap-bw
-%%DATADIR%%/gschem-colormap-darkbg
-%%DATADIR%%/gschem-colormap-lightbg
 %%DATADIR%%/icons/hicolor/16x16/actions/attribute-attach.png
 %%DATADIR%%/icons/hicolor/16x16/actions/attribute-detach.png
 %%DATADIR%%/icons/hicolor/16x16/actions/attribute-show-both.png
@@ -830,10 +908,16 @@
 %%DATADIR%%/icons/hicolor/22x22/actions/insert-symbol.png
 %%DATADIR%%/icons/hicolor/22x22/actions/insert-text.png
 %%DATADIR%%/icons/hicolor/22x22/actions/select.png
+%%DATADIR%%/geda-system.conf
+%%DATADIR%%/lepton-system.conf
+%%DATADIR%%/system-gafrc
+%%DATADIR%%/system-gschemrc
 %%DATADIR%%/lepton-gtkrc
-%%DATADIR%%/lepton-system.conf
 %%DATADIR%%/print-colormap-darkbg
 %%DATADIR%%/print-colormap-lightbg
+%%DATADIR%%/gschem-colormap-bw
+%%DATADIR%%/gschem-colormap-darkbg
+%%DATADIR%%/gschem-colormap-lightbg
 %%DATADIR%%/scheme/auto-place-attribs.scm
 %%DATADIR%%/scheme/auto-place-netname.scm
 %%DATADIR%%/scheme/auto-refdes.scm
@@ -908,8 +992,11 @@
 %%DATADIR%%/scheme/gschem/util.scm
 %%DATADIR%%/scheme/gschem/window.scm
 %%DATADIR%%/scheme/lepton/attrib.scm
+%%DATADIR%%/scheme/lepton/color-map.scm
 %%DATADIR%%/scheme/lepton/config.scm
 %%DATADIR%%/scheme/lepton/core/gettext.scm
+%%DATADIR%%/scheme/lepton/eval.scm
+%%DATADIR%%/scheme/lepton/ffi.scm
 %%DATADIR%%/scheme/lepton/file-system.scm
 %%DATADIR%%/scheme/lepton/legacy-config.scm
 %%DATADIR%%/scheme/lepton/legacy-config/keylist.scm
@@ -954,11 +1041,15 @@
 %%DATADIR%%/scheme/schematic/attrib.scm
 %%DATADIR%%/scheme/schematic/builtins.scm
 %%DATADIR%%/scheme/schematic/core/gettext.scm
+%%DATADIR%%/scheme/schematic/dialog.scm
+%%DATADIR%%/scheme/schematic/ffi.scm
+%%DATADIR%%/scheme/schematic/ffi/gtk.scm
 %%DATADIR%%/scheme/schematic/gschemdoc.scm
 %%DATADIR%%/scheme/schematic/gui/keymap.scm
 %%DATADIR%%/scheme/schematic/gui/stroke.scm
 %%DATADIR%%/scheme/schematic/hook.scm
 %%DATADIR%%/scheme/schematic/keymap.scm
+%%DATADIR%%/scheme/schematic/menu.scm
 %%DATADIR%%/scheme/schematic/netlist.scm
 %%DATADIR%%/scheme/schematic/precompile.scm
 %%DATADIR%%/scheme/schematic/repl.scm
@@ -995,6 +1086,87 @@
 %%DATADIR%%/scheme/symcheck/check.scm
 %%DATADIR%%/scheme/symcheck/option.scm
 %%DATADIR%%/scheme/symcheck/report.scm
+%%NLS%%share/locale/af/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/af/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/ar/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/ar/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/ar/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/bg/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/bg/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/bs/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/bs/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/ca/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/ca/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/da/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/da/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/de/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/de/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/de/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/el/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/el/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/el/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/en_GB/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/en_GB/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/en_GB/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/es/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/es/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/es/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/fa/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/fa/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/fi/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/fr/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/fr/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/fr/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/gl/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/he/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/hu/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/hu/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/hu/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/it/LC_MESSAGES/lepton-cli.mo
+%%NLS%%share/locale/it/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/it/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/it/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/ja/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/ja/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/ml/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/ml/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/nb/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/nl/LC_MESSAGES/lepton-cli.mo
+%%NLS%%share/locale/nl/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/nl/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/nl/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/oc/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/pl/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/pl/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/pl/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/pt/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/pt/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/pt/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/pt_BR/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/pt_BR/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/pt_BR/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/ru/LC_MESSAGES/lepton-cli.mo
+%%NLS%%share/locale/ru/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/ru/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/ru/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/sl/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/sl/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/sr/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/sr/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/sr/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/sv/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/sv/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/tr/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/tr/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/tr/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/uk/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/uk/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/zh_CN/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/zh_CN/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/zh_CN/LC_MESSAGES/libleptonattrib.mo
+%%NLS%%share/locale/zh_TW/LC_MESSAGES/lepton-schematic.mo
+%%NLS%%share/locale/zh_TW/LC_MESSAGES/liblepton.mo
+%%NLS%%share/locale/zh_TW/LC_MESSAGES/libleptonattrib.mo
 %%DATADIR%%/sym-gnetman/capacitor-1.sym
 %%DATADIR%%/sym-gnetman/ccvs-1.sym
 %%DATADIR%%/sym-gnetman/clkinv.sym
@@ -2483,91 +2655,3 @@
 %%DATADIR%%/sym/xilinx/XC2S50-5PQ208I.sym
 %%DATADIR%%/sym/xilinx/XC2S50-6PQ208C.sym
 %%DATADIR%%/sym/xilinx/XC95108-PC84.sym
-%%DATADIR%%/system-gafrc
-%%DATADIR%%/system-gschemrc
-%%NLS%%share/locale/af/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/af/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/ar/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/ar/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/ar/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/bg/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/bg/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/bs/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/bs/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/ca/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/ca/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/da/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/da/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/de/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/de/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/de/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/el/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/el/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/el/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/en_GB/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/en_GB/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/en_GB/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/es/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/es/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/es/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/fa/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/fa/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/fi/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/fr/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/fr/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/fr/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/gl/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/he/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/hu/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/hu/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/hu/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/it/LC_MESSAGES/lepton-cli.mo
-%%NLS%%share/locale/it/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/it/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/it/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/ja/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/ja/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/ml/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/ml/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/nb/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/nl/LC_MESSAGES/lepton-cli.mo
-%%NLS%%share/locale/nl/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/nl/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/nl/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/oc/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/pl/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/pl/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/pl/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/pt/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/pt/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/pt/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/pt_BR/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/pt_BR/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/pt_BR/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/ru/LC_MESSAGES/lepton-cli.mo
-%%NLS%%share/locale/ru/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/ru/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/ru/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/sl/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/sl/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/sr/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/sr/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/sr/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/sv/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/sv/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/tr/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/tr/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/tr/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/uk/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/uk/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/zh_CN/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/zh_CN/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/zh_CN/LC_MESSAGES/libleptonattrib.mo
-%%NLS%%share/locale/zh_TW/LC_MESSAGES/lepton-schematic.mo
-%%NLS%%share/locale/zh_TW/LC_MESSAGES/liblepton.mo
-%%NLS%%share/locale/zh_TW/LC_MESSAGES/libleptonattrib.mo
-share/mime/application/x-lepton-sch2pcb-project.xml
-share/mime/application/x-lepton-schematic.xml
-share/mime/application/x-lepton-symbol.xml
-share/mime/packages/liblepton.xml
-@dir %%DATADIR%%/sym/local
