diff --git a/cad/lepton-eda/Makefile b/cad/lepton-eda/Makefile
index eba7563005d1..6083f780df7e 100644
--- a/cad/lepton-eda/Makefile
+++ b/cad/lepton-eda/Makefile
@@ -1,7 +1,7 @@
 PORTNAME=	lepton-eda
-PORTVERSION=	1.9.16
+PORTVERSION=	1.9.17
 CATEGORIES=	cad
-MASTER_SITES=	https://github.com/lepton-eda/lepton-eda/releases/download/${PORTVERSION}-20210731/
+MASTER_SITES=	https://github.com/lepton-eda/lepton-eda/releases/download/${PORTVERSION}-20211219/
 
 MAINTAINER=	graahnul.grom@gmail.com
 COMMENT=	Lepton Electronic Design Automation
@@ -15,12 +15,24 @@ LIB_DEPENDS=	libfontconfig.so:x11-fonts/fontconfig \
 		libpng.so:graphics/png \
 		libstroke.so:devel/libstroke \
 		libgtkextra-x11-3.0.so:x11-toolkits/gtkextra \
-		libharfbuzz.so:print/harfbuzz
+		libharfbuzz.so:print/harfbuzz \
+		libffi.so:devel/libffi \
+		libunistring.so:devel/libunistring \
+		libgmp.so:math/gmp \
+		libltdl.so:devel/libltdl \
+		libfribidi.so:converters/fribidi \
+		libpcre.so:devel/pcre \
+		libgraphite2.so:graphics/graphite2 \
+		libexpat.so:textproc/expat2 \
+		libGLdispatch.so:graphics/libglvnd
 
 USES=		desktop-file-utils gettext-tools gnome \
 		groff:build libtool localbase makeinfo pathfix perl5 \
-		pkgconfig shared-mime-info shebangfix
-USE_GNOME=	cairo gtk20
+		pkgconfig shared-mime-info shebangfix iconv:wchar_t xorg
+USE_XORG=	xrender xinerama xi xrandr xcursor xext x11 \
+		xcomposite xdamage xfixes pixman xcb xau xdmcp
+USE_GNOME=	cairo gtk20 gdkpixbuf2
+USE_GL=	egl gl
 USE_LDCONFIG=	yes
 
 SHEBANG_FILES=	utils/pcb_backannotate/lepton-pcb_backannotate \
diff --git a/cad/lepton-eda/distinfo b/cad/lepton-eda/distinfo
index 0476c44befe7..6233f085cc22 100644
--- a/cad/lepton-eda/distinfo
+++ b/cad/lepton-eda/distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1631290180
-SHA256 (lepton-eda-1.9.16.tar.gz) = c5d275d4ce115cd17a78f643e98fbf0d474edb50052dbc9d65cabd9d6834f989
-SIZE (lepton-eda-1.9.16.tar.gz) = 9493429
+TIMESTAMP = 1640711244
+SHA256 (lepton-eda-1.9.17.tar.gz) = 94e9de29e254730ea3397ff78aff410d6389df18a9fef1a1cf11c73406edb12f
+SIZE (lepton-eda-1.9.17.tar.gz) = 9454425
diff --git a/cad/lepton-eda/pkg-plist b/cad/lepton-eda/pkg-plist
index 935c3fc22b76..aec945e034bc 100644
--- a/cad/lepton-eda/pkg-plist
+++ b/cad/lepton-eda/pkg-plist
@@ -1,17 +1,21 @@
 bin/lepton-archive
 bin/lepton-attrib
 bin/lepton-cli
+bin/lepton-config
 bin/lepton-embed
+bin/lepton-export
 bin/lepton-netlist
 bin/lepton-pcb_backannotate
 bin/lepton-refdes_renum
 bin/lepton-sch2pcb
 bin/lepton-schdiff
 bin/lepton-schematic
+bin/lepton-shell
 bin/lepton-symcheck
 bin/lepton-symfix
 bin/lepton-tragesym
 bin/lepton-upcfg
+bin/lepton-update
 include/liblepton/angle.h
 include/liblepton/arc.h
 include/liblepton/arc_object.h
@@ -33,8 +37,7 @@ include/liblepton/edaerrors.h
 include/liblepton/edapangorenderer.h
 include/liblepton/edapaths.h
 include/liblepton/edarenderer.h
-include/liblepton/edascmhookproxy.h
-include/liblepton/edascmvaluetypes.h
+include/liblepton/export.h
 include/liblepton/fill.h
 include/liblepton/forward.h
 include/liblepton/funcs.h
@@ -66,14 +69,14 @@ include/liblepton/transform.h
 include/liblepton/undo.h
 include/libleptonattrib/prototype.h
 lib/liblepton.so
-lib/liblepton.so.9
-lib/liblepton.so.9.0.0
+lib/liblepton.so.10
+lib/liblepton.so.10.0.0
 lib/libleptonattrib.so
-lib/libleptonattrib.so.3
-lib/libleptonattrib.so.3.0.0
+lib/libleptonattrib.so.4
+lib/libleptonattrib.so.4.0.0
 lib/libleptongui.so
-lib/libleptongui.so.3
-lib/libleptongui.so.3.0.1
+lib/libleptongui.so.4
+lib/libleptongui.so.4.0.0
 libdata/pkgconfig/liblepton.pc
 libdata/pkgconfig/libleptonattrib.pc
 libdata/pkgconfig/libleptongui.pc
@@ -91,6 +94,7 @@ man/man1/lepton-symcheck.1.gz
 man/man1/lepton-symfix.1.gz
 man/man1/lepton-tragesym.1.gz
 man/man1/lepton-upcfg.1.gz
+man/man1/lepton-update.1.gz
 share/applications/lepton-attrib.desktop
 share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/NEWS.md
@@ -101,11 +105,13 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Add-attributes.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Adding-attributes-to-your-symbol.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Allegro-backend.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Attribute-compatibility.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Attributes.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/BOM-backend.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Back-annotation-from-PADS-PowerPCB.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Box-selection.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Cascade-backend.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Check-symbols.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Colors.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Communication.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Compatibility-with-geda_002dgaf.html
@@ -134,6 +140,7 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/File-format-overview.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Filenames.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Forward-annotation-to-PADS-PowerPCB.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Hardcode-library-names-for-Guile.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Installation-from-git.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Installation-from-source.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Installation-of-lepton_002dattrib.html
@@ -147,8 +154,7 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Log-window.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Logging.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Macro-widget.html
-%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Master-Attribute-Document-Overview.html
-%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Master-Attribute-Document-Revision-History.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Master-attribute-list.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Menu-Add.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Menu-Attributes.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Menu-Edit.html
@@ -163,9 +169,12 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Missing-symbols.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/New-configuration-system.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/No_002dconnect-symbols.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Non_002dmenu-actions.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Object-types.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Obsolete-Attributes.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Obsolete-settings.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Other-resources.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Out_002dof_002dsource-build.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Overview.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/PADS-backend.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Path-data.html
@@ -177,6 +186,7 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Save-As-dialog-box.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Saving-schematic-pages.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Schematic-only-Attributes.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Searching-for-text.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Selecting-and-deselecting-all-objects.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Selecting-objects.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Simple-object-selection.html
@@ -195,30 +205,49 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Web-pages.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/What-are-Attributes_003f.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/Working-with-windows.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/action_002dfeedback_002dmode.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/always_002dpromote.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/analysis.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/arc.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/author-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/auto_002dsave_002dinterval.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/box.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/bus.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/bus_002dripper_002drotation.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/bus_002dripper_002dsize.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/bus_002dripper_002dsymname.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/bus_002dripper_002dtype.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/cache_002dconfiguration_002dcontext.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/circle.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/ckt.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/clocks.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/comment-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component_002dattributes.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component_002dlibrary.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component_002dlibrary_002dcommand.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component_002dlibrary_002dfuncs.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/component_002dlibrary_002dsearch.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/continue_002dcomponent_002dplace.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/coordinate-space.png
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/create_002dfiles.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/default_002dbus_002dname.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/default_002dfilename.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/default_002dnet_002dname.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/default_002dtitleblock.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/description-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/device-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/dist_002dlicense-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/documentation-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/dots_002dgrid_002ddot_002dsize.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/dots_002dgrid_002dfixed_002dthreshold.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/dots_002dgrid_002dmode.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/draw_002dgrips.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/email-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/embed_002dcomponents.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/enforce_002dhierarchy.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/export-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fast_002dmousepan.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003acascade1.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003acascade2.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003aswitcap1.html
@@ -227,16 +256,22 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003aswitcap4.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003aswitcap5.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/fig_003aswitcap6.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/file_002dpreview.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/font.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/footprint-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/force_002dboundingbox.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gEDA-File-Format-Document-Revision-History.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gEDA-file-format.html
-%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gEDA_002fgaf-Master-Attribute-Document.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gafrc.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/graphical-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/grid_002dmode.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/gschemrc.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/handleboxes.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/index.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/keep_002dinvisible.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/keyboardpan_002dgain.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/label-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/layout.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton-manual.css
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dattrib-menus.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dattrib.html
@@ -251,17 +286,38 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dschematic-configuration.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dschematic-window.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dschematic.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/lepton_002dupdate.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/line.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/log_002dwindow.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/logging.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/magnetic_002dnet_002dmode.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/mangle_002dnet_002dattribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/mangle_002dnetname_002dattribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/mangle_002drefdes_002dattribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/max_002drecent_002dfiles.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/mesh_002dgrid_002ddisplay_002dthreshold.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/middle_002dbutton.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/monochrome.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/mousepan_002dgain.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/name-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net-attribute.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net_002dattribute_002dorder.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net_002dattribute_002dseparator.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net_002dconsolidate.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net_002ddirection_002dmode.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net_002dnaming_002dpriority.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net_002dselection_002dmode.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/net_003d-attribute-mini_002dHOWTO.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/netconn_002drubberband.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/netlist-group.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/netlist_002ehierarchy-group.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/netname-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/netname_002dattribute_002dorder.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/netname_002dattribute_002dseparator.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/numslots-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/paper.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/path-example.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/path.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/picture.html
@@ -272,9 +328,13 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/pinseq-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/pintype-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/project_002dconfiguration.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/promote.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/promote_002dinvisible.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/refdes-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/refdes-naming-conventions.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/refdes-naming-restrictions.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/refdes_002dattribute_002dorder.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/refdes_002dattribute_002dseparator.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/reset_002dcomponent_002dlibrary.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/rf_cascade.png
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic-group.html
@@ -288,23 +348,42 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002estatus_002dbar-group.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002etabs-group.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/schematic_002eundo-group.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/scroll_002dwheel.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/scrollbars.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/scrollpan_002dsteps.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/select_002dslack_002dpixels.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/slot-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/slot_0023-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/slotdef-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/snap_002dsize.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/sort.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/source-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/symversion-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/table_003afonts.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/table_003apintype.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/text-and-attributes.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/text-layout.png
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/text_002dcaps_002dstyle.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/text_002dsize.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/third_002dbutton.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/third_002dbutton_002dcancel.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/toolbars.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/traverse_002dhierarchy.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/type-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/undo_002dcontrol.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/undo_002dlevels.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/undo_002dpanzoom.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/undo_002dtype.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/uref-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/use_002ddocks.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/use_002dlicense-attribute.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/use_002dtabs.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/user_002dconfiguration.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/value-attribute.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/version.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/warp_002dcursor.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/zoom_002dgain.html
+%%PORTDOCS%%%%DOCSDIR%%/lepton-manual.html/zoom_002dwith_002dpan.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Actions.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Arcs.html
 %%PORTDOCS%%%%DOCSDIR%%/lepton-scheme.html/Attributes.html
@@ -368,25 +447,23 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-symfix.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-tragesym.html
 %%PORTDOCS%%%%DOCSDIR%%/man/lepton-upcfg.html
+%%PORTDOCS%%%%DOCSDIR%%/man/lepton-update.html
 %%PORTDOCS%%%%DOCSDIR%%/readmes/README.bom
 %%PORTDOCS%%%%DOCSDIR%%/readmes/README.eagle
 %%PORTDOCS%%%%DOCSDIR%%/readmes/README.lepton-sch2pcb
 %%PORTDOCS%%%%DOCSDIR%%/readmes/README.lepton-tragesym
 %%PORTDOCS%%%%DOCSDIR%%/readmes/README.pcb
-%%PORTDOCS%%%%DOCSDIR%%/readmes/README.switcap
 %%PORTDOCS%%%%DOCSDIR%%/readmes/README.sysc
 %%PORTDOCS%%%%DOCSDIR%%/readmes/README.verilog
 %%PORTDOCS%%%%DOCSDIR%%/readmes/README.vhdl
 %%PORTDOCS%%%%DOCSDIR%%/readmes/syntax_architecture.txt
 %%PORTDOCS%%%%DOCSDIR%%/readmes/syntax_entity.txt
 %%PORTDOCS%%%%DOCSDIR%%/readmes/vams_mode.txt
-%%PORTDOCS%%%%DOCSDIR%%/wiki/best_practices.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/design.css
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-add_a_ground_plane_to_your_layout.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-bom_readme.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-bug_triage_guide.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-circuit_simulation_improvements.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-covered_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-csygas.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-csygas.ru.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-data_plotting_improvements.html
@@ -413,8 +490,6 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gaf_building_git_version.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gaf_utility.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gattrib_readme.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gedasuite_installation.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gerbv_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gerbv_pnp_readme.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gfdl.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-glossary.html
@@ -437,7 +512,6 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gparts_ii.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gparts_tc.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gparts_ug.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-grenum_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gsch2pcb_readme.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gsch2pcb_tutorial.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gsch2pcb_tutorial_releasenotes.html
@@ -468,7 +542,6 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gschem_ug-viewing_schematics.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gschem_ug.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gschem_warmup.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-gsymcheck_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-guile_scripting.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-hierarchy.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-howto_report_bugs.html
@@ -478,13 +551,10 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-icarus_extensions.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-icarus_glossary.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-icarus_ieee1364.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-icarus_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-icarus_opcodes.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-icarus_quick_start.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-icarus_readme.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-icarus_vpi_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-icarus_vpi_within_vvp.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-icarus_vvp_runtime.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-icarus_vvp_simulation.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-icarus_xilinx_hints.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-icarus_xnf.html
@@ -495,10 +565,7 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-links.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-mailinglists.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-mcalc_readme.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-ngnutmeg_mp.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-ngsconvert_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-ngspice_and_gschem.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-ngspice_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-olib_readme.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-pcb-quick_reference.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-pcb_developer_introduction.html
@@ -508,18 +575,11 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-pcb_funding_sow-gui_ideas.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-pcb_funding_sow.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-pcb_layers.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-pcb_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-pcb_tips.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-release_engineering.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-scm.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-screenshots.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-sdb_howto.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-sn_readme.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-spice_improvements.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-spice_polys.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-start.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-style_guide.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-syntax_features.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-systemc_netlister_readme.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-tasks.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-todos.html
@@ -532,21 +592,14 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-verilog_netlister_readme.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-version_control_migration.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-vhdl_netlister_readme.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-wcalc_mp.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-wcalc_readme.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/geda-wcalc_stdio_mp.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-wip-clipboard.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/geda-xml_file_formats.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/gerbv-projects.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/glue-projects.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/gschem-projects.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/gsoc2007.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/gsoc2007_projects.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/gsoc2008.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/gsoc2008_projects.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/gsoc2009.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/gsoc2009_projects.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/gsoc2010.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/gsoc2010_projects.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/gtkwave-projects.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/gwave-projects.html
@@ -598,7 +651,6 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/board-4.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/board-5.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/board-6.png
-%%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/coordinatespace.jpg
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/coordinatespace.ru.png
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/design_flow.jpg
 %%PORTDOCS%%%%DOCSDIR%%/wiki/media/geda/design_flow.ru.png
@@ -686,9 +738,6 @@ share/applications/lepton-schematic.desktop
 %%PORTDOCS%%%%DOCSDIR%%/wiki/sitemap.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/start.html
 %%PORTDOCS%%%%DOCSDIR%%/wiki/style.css
-%%PORTDOCS%%%%DOCSDIR%%/wiki/wiki-dokuvimki.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/wiki-dokuwiki.html
-%%PORTDOCS%%%%DOCSDIR%%/wiki/wiki-syntax.html
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/README
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/5980-2396E.pdf
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/RF_Amp/MSA-2643.cir
@@ -978,6 +1027,7 @@ share/info/text-layout.png
 %%DATADIR%%/scheme/lepton/core/gettext.scm
 %%DATADIR%%/scheme/lepton/eval.scm
 %%DATADIR%%/scheme/lepton/ffi.scm
+%%DATADIR%%/scheme/lepton/ffi/lib.scm
 %%DATADIR%%/scheme/lepton/file-system.scm
 %%DATADIR%%/scheme/lepton/legacy-config.scm
 %%DATADIR%%/scheme/lepton/legacy-config/keylist.scm
@@ -985,12 +1035,14 @@ share/info/text-layout.png
 %%DATADIR%%/scheme/lepton/library/component.scm
 %%DATADIR%%/scheme/lepton/log-rotate.scm
 %%DATADIR%%/scheme/lepton/log.scm
+%%DATADIR%%/scheme/lepton/m4.scm
 %%DATADIR%%/scheme/lepton/object.scm
 %%DATADIR%%/scheme/lepton/object/foreign.scm
 %%DATADIR%%/scheme/lepton/object/type.scm
 %%DATADIR%%/scheme/lepton/option.scm
 %%DATADIR%%/scheme/lepton/os.scm
 %%DATADIR%%/scheme/lepton/page.scm
+%%DATADIR%%/scheme/lepton/page/foreign.scm
 %%DATADIR%%/scheme/lepton/rc.scm
 %%DATADIR%%/scheme/lepton/repl.scm
 %%DATADIR%%/scheme/lepton/srfi-37.scm
